<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgColor="#ffffff"><table width="600" border="0" cellspacing="0" cellpadding="0"><p><b>Chapter 19:&nbsp;Summary of Key Concepts</b></p><SPAN CLASS="Header">EUKARYOTIC CHROMATIN STRUCTURE</SPAN><BR><UL><LI><a name="para09"></a><p class="BlockText"><span class="Emphasis2">Chromatin structure is based on successive levels of DNA packing (pp. 354-356,</span>&nbsp;<span class="SmallCap">FIGURE 19.1</span><span class="Emphasis2">)</span> Eukaryotic chromatin is composed mostly of DNA and histone proteins that bind to the DNA to form nucleosomes, the most basic units of DNA packing. Additional folding leads ultimately to highly compacted heterochromatin, the form of chromatin in a metaphase chromosome. In interphase cells, most chromatin is in a highly extended form, euchromatin.</p><a name="para10"></a><p class="BlockText"><span class="Emphasis2">Web/CD Activity19A:<span class="Emphasis1">
			
			&nbsp;<A class="media" href="javascript:bcPopActivity('19A')">DNA Packing</A></span></span></p></LI></UL><SPAN CLASS="Header">GENOME ORGANIZATION AT THE DNA LEVEL</SPAN><BR><UL><LI><a name="para11"></a><p class="BlockText"><span class="Emphasis2">Repetitive DNA and other noncoding sequences account for much of a eukaryotic genome (pp. 357-358,</span>TABLE 19.1<span class="Emphasis2">)</span> Sequences of up to ten nucleotide pairs tandemly repeated thousands of times are especially prominent in the DNA of centromeres and telomeres, where they probably play structural roles in the chromosome. The units of interspersed repetitive DNA are typically longer, and most may be transposons or retrotransposons.</p></LI><LI><a name="para12"></a><p class="BlockText"><span class="Emphasis2">Gene families have evolved by duplication of ancestral genes (pp. 358-359,</span>&nbsp;<span class="SmallCap">FIGURE 19.3</span><span class="Emphasis2">)</span> The ribosomal RNA genes, a tandemly arranged family of identical genes, enable the cell to make millions of ribosomes quickly. The nonidentical genes in the two globin gene families encode polypeptides used at different developmental stages of the animal.</p></LI><LI><a name="para13"></a><p class="BlockText"><span class="Emphasis2">Gene amplification, loss, or rearrangement can alter a cell’s genome during an organism’s lifetime (pp. 359-362,</span>&nbsp;<span class="SmallCap">FIGURES 19.5, 19.6</span><span class="Emphasis2">)</span> The selective replication of certain genes, such as rRNA genes, occurs during development in some species, further bolstering the cells’ ability to make rRNA in large amounts. Some species selectively lose entire chromosomes or parts of chromosomes in certain cells. Rearrangements of DNA in a somatic cell by transposons or retrotransposons can alter the cell’s genome in ways that affect gene expression. In vertebrates, a rearrangement and selective deletion of DNA segments in differentiating B lymphocytes accounts for antibody diversity.</p><a name="para14"></a><p class="BlockText"><span class="Emphasis2">Web/CD Activity19B:<span class="Emphasis1">
			
			&nbsp;<A class="media" href="javascript:bcPopActivity('19B')">Gene Amplification, Loss, and Rearrangement</A></span></span></p></LI></UL><SPAN CLASS="Header">THE CONTROL OF GENE EXPRESSION</SPAN><BR><UL><LI><a name="para15"></a><p class="BlockText"><span class="Emphasis2">Each cell of a multicellular eukaryote expresses only a small fraction of its genes (p. 362)</span> In particular, selective control of genes is required for cellular differentiation.</p></LI><LI><a name="para16"></a><p class="BlockText"><span class="Emphasis2">The control of gene expression can occur at any step in the pathway from gene to functional protein: <span class="Emphasis1">an overview</span> (p. 362,</span>&nbsp;<span class="SmallCap">FIGURE 19.7</span><span class="Emphasis2">)</span> Key stages include chromatin changes, initiation of transcription, RNA processing, translation, modification of the protein, and degradation of mRNA and protein.</p><a name="para17"></a><p class="BlockText"><span class="Emphasis2">Web/CD Activity19C:<span class="Emphasis1">
			
			&nbsp;<A class="media" href="javascript:bcPopActivity('19C')">Overview: Control of Gene Expression</A></span></span></p></LI><LI><a name="para18"></a><p class="BlockText"><span class="Emphasis2">Chromatin modifications affect the availability of genes for transcription (pp. 362-364)</span> DNA methylation seems to diminish transcription of that DNA. Histone acetylation seems to loosen nucleosome structure and thereby enhance transcription.</p></LI><LI><a name="para19"></a><p class="BlockText"><span class="Emphasis2">Transcription initiation is controlled by proteins that interact with DNA and with each other (pp. 364-367,</span>&nbsp;<span class="SmallCap">FIGURES 19.8</span>, <span class="SmallCap">19.9</span><span class="Emphasis2">)</span> Multiple DNA control elements that bind specific transcription factors regulate the formation of a transcription initiation complex at a eukaryotic promoter. But protein-protein interactions are as important as protein-DNA binding in regulating eukaryotic genes. Bending of DNA enables transcription factors bound to control elements (enhancers) distant from the promoter to contact proteins at the promoter. Unlike the genes of a prokaryotic operon, coordinately controlled eukaryotic genes each have a promoter and control elements; the same regulatory sequences are common to all the genes of a group, enabling recognition by the same transcription factors.</p><a name="para20"></a><p class="BlockText"><span class="Emphasis2">Web/CD Activity19D:<span class="Emphasis1">
			
			&nbsp;<A class="media" href="javascript:bcPopActivity('19D')">Control of Transcription</A></span></span></p><a name="para21"></a><p class="BlockText"><span class="Emphasis2">Web/CD Case Study in the Process of Science: <span class="Emphasis1">
			
			&nbsp;<A class="media" href="javascript:openExpr('../../interactivemedia/activities/HPOS19/HPOS1901/index.html')">How Do You Design a Gene Expression System?</A></span></span></p></LI><LI><a name="para22"></a><p class="BlockText"><span class="Emphasis2">Post-transcriptional mechanisms play supporting roles in the control of gene expression (pp. 367-368,</span>&nbsp;<span class="SmallCap">FIGURES 19.11</span>, <span class="SmallCap">19.12</span><span class="Emphasis2">)</span> Regulation at the RNA-processing level is exemplified by alternative RNA splicing. Also, each mRNA has a characteristic life span, which sequences in the leader and trailer regions help determine. The initiation of translation can be controlled via regulation of initiation factors. After translation, various types of protein processing (such as cleavage and the addition of chemical groups) are subject to control, as is the degradation of the protein.</p><a name="para23"></a><p class="BlockText"><span class="Emphasis2">Web/CD Activity19E:<span class="Emphasis1">
			
			&nbsp;<A class="media" href="javascript:bcPopActivity('19E')">Post-Transcriptional Control Mechanisms</A></span></span></p><a name="para24"></a><p class="BlockText"><span class="Emphasis2">Web/CD Activity19F:<span class="Emphasis1">
			
			&nbsp;<A class="media" href="javascript:bcPopActivity('19F')">Review: Control of Gene Expression</A></span></span></p></LI></UL><SPAN CLASS="Header">THE MOLECULAR BIOLOGY OF CANCER</SPAN><BR><UL><LI><a name="para25"></a><p class="BlockText"><span class="Emphasis2">Cancer results from genetic changes that affect the cell cycle (p. 369,</span>&nbsp;<span class="SmallCap">FIGURE 19.13</span><span class="Emphasis2">)</span> The products of proto-oncogenes and tumor-suppressor genes control cell division. A DNA change that makes a proto-oncogene excessively active converts it to an oncogene, which may promote excessive cell growth and cancer. A tumor-suppressor gene encodes a protein that inhibits abnormal cell division. The loss or mutation of such a gene has effects similar to the activation of an oncogene.</p></LI><LI><a name="para26"></a><p class="BlockText"><span class="Emphasis2">Oncogene proteins and faulty tumor-suppressor proteins interfere with normal signaling pathways (pp. 369-371,</span>&nbsp;<span class="SmallCap">FIGURE 19.14</span><span class="Emphasis2">)</span> Typical components of both growth-stimulating and growth-inhibiting pathways include growth factors, membrane receptors, G proteins (such as Ras), protein kinases, and transcription activators. A hyperactive version of a protein in a stimulatory pathway, such as Ras (a G protein), functions as an oncogene protein. A defective version of a protein in an inhibitory pathway, such as p53 (a transcription activator), fails to function as a tumor suppressor.</p></LI><LI><a name="para27"></a><p class="BlockText"><span class="Emphasis2">Multiple mutations underlie the development of cancer (pp. 371-372,</span>&nbsp;<span class="SmallCap">FIGURE 19.15</span><span class="Emphasis2">)</span> Tumor cells accumulate changes affecting proto-oncogenes and tumor-suppressor genes. Some of these mutations can be inherited, resulting in a predisposition to developing certain types of cancer.</p><a name="para28"></a><p class="BlockText"><span class="Emphasis2">Web/CD Activity19G:<span class="Emphasis1">
			
			&nbsp;<A class="media" href="javascript:bcPopActivity('19G')">Causes of Cancer</A></span></span></p></LI></UL></table></body></html>
