<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Repetitive DNA and other noncoding sequences account for much of a eukaryotic genome</p><a name="para01"></a><p class="BlockText">In prokaryotes, most of the DNA in a genome codes for protein (or tRNA and rRNA), with the small amount of noncoding DNA consisting mainly of regulatory sequences, such as promoters. The coding sequence of nucleotides along a prokaryotic gene proceeds from start to finish without interruption. In eukaryotic genomes, by contrast, most of the DNA--about 97%  in humans--does <span class="Emphasis1">not</span> encode protein or RNA. What does this DNA consist of? Some of it is known to be regulatory sequences, but much of it consists of sequences whose functions, if any, are not yet understood. This DNA includes introns, the stretches of noncoding DNA that often interrupt the coding sequences of eukaryotic genes (see <span class="SmallCap">FIGURE 17.9</span>). Even more of the noncoding DNA consists of <span class="Emphasis2">repetitive DNA,</span> nucleotide sequences that are present in many copies in a genome, usually not within genes.</p><p class="Header3">Tandemly Repetitive DNA</p><a name="para02"></a><p class="BlockText">TABLE 19.1 lists the main categories of repetitive DNA, which occur in different amounts in different species. In mammals, about 10-15%  of the genome is <span class="Emphasis1">tandemly repetitive DNA,</span> short sequences repeated in series, as in the following example (showing one DNA strand only):&#160;...&#160; GTTACGTTACGTTACGTTACGTTACGTTAC&#160;...&#160; The number of repetitions of the GTTAC unit at a site in the genome could be as high as several hundred thousand. Repeated units are up to ten base pairs long.</p><TABLE BORDER="0"><TR><TH BGCOLOR="#f1f1f1"><B>Table 19-1.</B>&nbsp;<span class="table_title">Types of Repetitive DNA</span><BR></TH></TR><TR><TD><TABLE BORDER="0" CELLPADDING="5" CELLSPACING="5"><TR><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN=""><span class="Emphasis2">Tandemly Repetitive DNA (Satellite DNA)</span></TH><TH BGCOLOR="#f1f1f1" COLSPAN="" ROWSPAN=""></TH></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Repeated units at a site are usually identical</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Proportion of mammalian DNA:</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">10-15% </P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Length of each repeated unit:</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">1-10 base pairs</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Total length of repetitive DNA per site, in base pairs:</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Regular satellite DNA</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">100,000-10 million</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Minisatellite DNA</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">100-100,000</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Microsatellite DNA</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">10-100</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"><span class="Emphasis2">Interspersed Repetitive DNA</span></P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">"Copies" are very similar but not identical</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText"></P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Proportion of mammalian DNA:</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">25-40% </P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Length of each repeated unit:</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">100-10,000 base pairs</P></TD></TR><TR><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">Number of repetitions per genome:</P></TD><TD valign="top" COLSPAN="" ROWSPAN=""><P class="NormText">10-1 million</P></TD></TR></TABLE></TD></TR></TABLE><a name="para27"></a><p class="BlockText">The nucleotide composition of tandemly repetitive DNA is often different enough from the rest of the cell’s DNA to give the repetitive DNA an intrinsically different density, enabling researchers to isolate it by differential ultracentrifugation (as in the Meselson-Stahl experiment; see <span class="SmallCap">FIGURE 16.9</span>). If the genomic DNA is cut into pieces before centrifugation, segments of different density migrate to different positions in the centrifuge tube. Repetitive DNA isolated in this way was originally called <span class="Emphasis2">satellite DNA,</span> because it appeared as a "satellite" band in the centrifuge tube, separate from the rest of the DNA. Now the term is often used for all tandemly repetitive DNA.</p><a name="para28"></a><p class="BlockText">As TABLE 19.1 shows, satellite DNA is classified into three types, depending on the total length of the DNA at each site. Regular satellite DNA occurs in stretches over 100,000 base pairs long, while <span class="Emphasis1">minisatellite</span> and <span class="Emphasis1">microsatellite</span> subcategories have been created for sequences that appear in shorter stretches. Microsatellite DNA, with short units repeated only 10 to 100 times, has turned out to be extremely useful for DNA fingerprinting, as we’ll discuss in Chapter 20.</p><a name="para29"></a><p class="BlockText">A number of genetic disorders, including two you read about in Chapter 15, are caused by abnormally long stretches of tandemly repeated nucleotide triplets--a sort of microsatellite--within the affected gene. The first such "triplet repeat" sequence to be identified was the one responsible for fragile X syndrome, a major cause of mental retardation. In the normal allele for the fragile X gene, within the 5<B>&#39;</B> untranslated leader region of the first exon, the triplet CGG is repeated about 30 times; in a fragile X allele, the same triplet is repeated hundreds or even thousands of times, creating the fragile site. The lengthening of the repetitive DNA occurs in steps over a number of generations. Huntington’s disease is also a triplet-repeat disorder, with a stretch of repeated CAG triplets that lengthens in a similar fashion. In this case, the triplet repeat is actually translated, and the resulting protein has a long string of glutamines, the amino acid encoded in DNA by CAG (recall that DNA sequences are given as they appear on the <span class="Emphasis1">non</span>template strand). The dozen triplet-repeat disorders identified to date all affect the nervous system, and in all cases the number of repeats seems to correlate with the severity of the disease and the age of onset. Scientists do not yet know exactly how the extra repeats cause disorders; the mechanism undoubtedly varies with the gene.</p><a name="para30"></a><p class="BlockText">Much of a genome’s regular satellite DNA is located at chromosomal telomeres and centromeres, suggesting that this DNA plays a structural role for chromosomes. The DNA at centromeres is essential for the separation of chromatids in cell division (see Chapter 12), and along with other tandemly repetitive sequences, it may help organize the chromatin within the interphase nucleus. We discussed the DNA of telomeres, the tips of chromosomes, in Chapter 16. Besides protecting genes from being lost as the DNA shortens with each round of replication, telomeric DNA protects the chromosome by binding proteins that protect the ends from degradation and from joining to other chromosomes.</p><a name="para31"></a><p class="BlockText">The importance of telomeres and centromeres has been highlighted by the creation of <span class="Emphasis1">artificial chromosomes</span> in the laboratory. The DNA used to make such chromosomes must, of course, include an origin of replication. But this is not sufficient; to function as a chromosome, it must also have a centromere and two telomeres. If an artificial chromosome has these features--no matter what the nature of its other DNA--a cell will be able to replicate it and distribute the copies properly to daughter cells.</p><p class="Header3">Interspersed Repetitive DNA</p><a name="para32"></a><p class="BlockText">In addition to tandemly repetitive DNA, eukaryotic genomes have huge amounts of <span class="Emphasis1">interspersed repetitive DNA</span> (see TABLE 19.1). The repeated units of this type of DNA are not next to each other; instead, they are scattered about the genome. A single unit is usually hundreds or even thousands of base pairs long, and the dispersed "copies" are similar but usually not identical to each other. Interspersed repetitive DNA makes up 25-40%  of most mammalian genomes. In humans and other primates, a large portion of this DNA (at least 5%  of the genome) consists of a family of similar sequences called <span class="Emphasis2"><span class="Emphasis1">Alu</span> elements,</span> for which each unit is about 300 nucleotide pairs long. Like the Huntington’s disease triplet repeats, <span class="Emphasis1">Alu</span> elements are an exception to the notion that repetitive DNA is noncoding. Many <span class="Emphasis1">Alu</span> elements are transcribed into RNA molecules; their cellular function, if any, is unknown.</p><a name="para33"></a><p class="BlockText">Although we know little about the functions of interspersed repetitive DNA, we do know how it comes to be both abundant and variable in location. Most such sequences, at least in mammals, seem to be transposons, the mobile genetic elements introduced in Chapter 18. We’ll have more to say about eukaryotic transposons later in this chapter.</p></td></tr></table></body></html>
