<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Modern phylogenetic systematics is based on cladistic analysis</p><a name="para01"></a><p class="BlockText">Classification based on evolutionary history is called phylogenetic systematics. So far, we have assumed that the phylogeny of a group of species is already known, and then we have applied that knowledge to a hierarchical classification of the species (see <span class="SmallCap">FIGURE 25.8</span>). But how do we reconstruct evolutionary history to draw phylogenetic trees in the first place? The fossil record helps, of course. Assessing relationships between living species by comparing their anatomy also provides data. And comparing two species’ DNA (or proteins) gets to their hereditary relationships at the molecular level. But how do systematists evaluate all this information and apply it to tree building and classification? Today, most systematists practice what is called cladistic analysis, or simply cladistics. We can trace the seeds of this revolution in systematics to a book that German entomologist (insect specialist) Willi Hennig wrote 50 years ago. Let’s examine some of the key elements of cladistic analysis.</p><p class="Header3">Clades: Monophyletic Groups</p><a name="para02"></a><p class="BlockText">A phylogenetic diagram based on cladistics is called a <span class="Emphasis2">cladogram.</span> It is a tree constructed from a series of dichotomies, or two-way branch points. Each branch point represents the divergence of two species from a common ancestor. For example, we could represent a branching within the cat family this way: <table border="0" cellspacing="0" cellpadding="5"><tr><td valign="top" align="center"><img src="../img/ufg/UFIG25-2.jpg" border="0" alt="" name="myUFIG25-2.jpg"></td></tr><tr><td class="figure_caption" bgcolor="#f1f1f1"></td></tr></table> We can also diagram dichotomous branching of taxa more inclusive than species, such as families or orders: <table border="0" cellspacing="0" cellpadding="5"><tr><td valign="top" align="center"><img src="../img/ufg/UFIG25-3.jpg" border="0" alt="" name="myUFIG25-3.jpg"></td></tr><tr><td class="figure_caption" bgcolor="#f1f1f1"></td></tr></table></p><a name="para03"></a><p class="BlockText">The "deeper" branch point represents evolutionary divergence from an ancestor common to both the cat and dog families. The sequence of branching symbolizes historical chronology; the last ancestor common to both the cat and dog families lived longer ago than the last ancestor shared by leopards and domestic cats.</p><a name="para04"></a><p class="BlockText">Each evolutionary branch in a cladogram is called a <span class="Emphasis2">clade</span> (from the Greek <span class="Emphasis1">clados</span>, branch). Notice that clades, like taxonomic levels, can be nested within larger clades. For example, the cat family represents a clade within a larger clade that also includes the dog family. But not all groupings of organisms qualify as clades. A clade consists of an ancestral species and all of its descendants. Such a group of species, be it a genus, family, or some higher taxa, is said to be <span class="Emphasis2">monophyletic,</span> which means "single tribe."<span class="SmallCap">FIGURE 25.9</span> contrasts a monophyletic group, or clade, with groupings of species that are unacceptable in the practice of cladistics.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig25-9','../fig/big/fig25-9.jpg','fig25-9');"><img border="0" src="../fig/thm/fig25-9.jpg" alt="fig25-9" name="myfig25-9"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 25-9.</B>&nbsp;<span><span class="Emphasis2">Monophyletic versus paraphyletic and polyphyletic groups.</span></span></td></tr></table><p class="Header3">Constructing a Cladogram</p><a name="para05"></a><p class="BlockText">How do systematists decide on the sequence of branching in a cladogram? Put another way, how can we determine the similarities between species that are relevant for grouping the species as a clade? This can be quite a challenge. The first complication is that not all similarity represents common ancestry.</p><a name="para06"></a><p class="BlockText"><span class="Emphasis2">Sorting Homology from Analogy.</span> Recall from Chapter 22 that likeness attributed to shared ancestry is called <span class="Emphasis2">homology.</span> The forelimbs of mammals are homologous; that is, the similarity in the intricate skeleton that supports the limbs has a genealogical basis (see <span class="SmallCap">FIGURE 22.14</span>).</p><a name="para07"></a><p class="BlockText">There is a wild card in this game of making evolutionary connections by evaluating similarity: Not all likeness qualifies as homology. Species from different evolutionary branches may come to resemble one another if they have similar ecological roles and natural selection has shaped analogous adaptations. This is called <span class="Emphasis2">convergent evolution,</span> and similarity due to convergence is called <span class="Emphasis2">analogy,</span> not homology (<span class="SmallCap">FIGURE 25.10</span>). The wings of bats and those of birds, for example, are analogous flight equipment. The fossil record documents that bat wings and bird wings evolved independently from walking forelimbs of different ancestors. (Note that homology, like taxonomy, can be hierarchical; though the forelimbs of bats and birds are analogous as <span class="Emphasis1">wings</span>, they are homologous as forelimbs descendant from the basic vertebrate prototype limb.)</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig25-10','../fig/big/fig25-10.jpg','fig25-10');"><img border="0" src="../fig/thm/fig25-10.jpg" alt="fig25-10" name="myfig25-10"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 25-10.</B>&nbsp;<span><span class="Emphasis2">Convergent evolution and analogous structures. (a)</span> The ocotillo of southwestern North America looks remarkably similar to <span class="Emphasis2">(b)</span> the allauidia found in Madagascar. The plants are not closely related and owe their resemblance to analogous adaptations that evolved independently in response to similar environmental pressures.</span></td></tr></table><a name="para08"></a><p class="BlockText">As a general rule, the greater the number of homologous parts between two species, the more closely the species are related, and this should be reflected in their classification. This guideline is simpler in principle than it is in practice. Adaptation can obscure homologies, and convergence can create misleading analogies. As we saw in Chapter 22, comparing the embryonic development of the features in question can often expose homology that is not apparent in the mature structures.</p><a name="para09"></a><p class="BlockText">There is another clue to identifying homology and sorting it from analogy: The more complex two similar structures are, the less likely it is they evolved independently. Consider the skulls of a human and a chimpanzee, for example. The skulls are not single bones, but a fusion of many, and the two skulls match almost perfectly, bone for bone. It is highly improbable that such complex structures matching in so many details could have separate origins. Most likely, the genes required to build these skulls were inherited from a common ancestor.</p><a name="para10"></a><p class="BlockText"><span class="Emphasis2">Identifying Shared Derived Characters.</span> Beyond separating homologous from analogous similarity, systematists must sort through the homologies to identify what are called shared derived characters. "Character" here refers to any feature that a particular taxon possesses. The characters that are relevant to phylogeny, of course, are the homologous ones. For example, hair is a character shared by all mammals. But so is a backbone. The presence of a backbone cannot help us distinguish mammals from other vertebrates because nonmammalian vertebrates such as fishes and reptiles also have backbones. In other words, the backbone is a homology that predates the branching of the mammalian clade from the vertebrate tree; it is a <span class="Emphasis2">shared primitive character,</span> a homology common to a taxon more inclusive than the one we’re trying to define. In contrast, hair is a homology found only in the vertebrates called mammals; in comparing mammals to other vertebrates, hair is a <span class="Emphasis2">shared derived character,</span> meaning an evolutionary novelty unique to a particular clade. The following flowchart summarizes the types of similarities between species: <table border="0" cellspacing="0" cellpadding="5"><tr><td valign="top" align="center"><img src="../img/ufg/UFIG25-4.jpg" border="0" alt="" name="myUFIG25-4.jpg"></td></tr><tr><td class="figure_caption" bgcolor="#f1f1f1"></td></tr></table> Note that the backbone can also qualify as a shared derived character, but at a deeper branch point that distinguishes <span class="Emphasis1">all</span> vertebrates from other animals. <span class="Emphasis1">Among</span> vertebrates, the backbone is a shared primitive character because it evolved in the ancestor common to all vertebrates. Thus, the sequence of branching in a cladogram represents the sequence in which evolutionary novelties, or shared derived characters, evolved.</p><a name="para11"></a><p class="BlockText"><span class="Emphasis2">Performing Outgroup Comparison.</span> As a key step in cladistic analysis, systematists use a method called outgroup comparison to differentiate shared characters that are derived from those that are primitive. Let’s work with the simplified case of ordering five vertebrates--a leopard, a turtle, a salamander, a tuna, and a creature called a lamprey--into a cladogram. As a basis of comparison, we need to designate an <span class="Emphasis2">outgroup,</span> which is a species or group of species that is closely related to the species we are studying, but known to be less closely related than any study-group members are to each other (based on evidence from paleontology, analysis of embryonic development, and molecular biology, for example). A good choice of an outgroup in our example is an animal called a lancelet. The five vertebrates we are studying make up the <span class="Emphasis2">ingroup,</span> and we can begin building our cladogram by comparing this ingroup with the outgroup.</p><a name="para12"></a><p class="BlockText">Outgroup comparison is based on the assumption that homologies present in both the outgroup and the ingroup must be primitive characters that were already present in the ancestor common to both groups. In our study case, an example of such a character is a structure called a notochord, a flexible rod running the length of the animal. (In vertebrates, the notochord is present in embryos, but the only vestiges of the structure in adults of many vertebrates are the rubbery disks between the vertebrae of the backbone.) The species making up the ingroup display a mixture of shared primitive and shared derived characters. The outgroup comparison enables us to focus on just those characters that were derived at the various branch points of vertebrate evolution. <span class="SmallCap">FIGURE 25.11a</span> tabulates examples of these characters. Note that <span class="Emphasis1">all</span> the vertebrates in the ingroup have backbones; thus, this is a shared primitive character that was present in the ancestral vertebrate, though not in the outgroup. Now note that the presence of jaws is a character absent in lampreys but present in other members of the ingroup. Thus, this character helps us identify the earliest branch point in the vertebrate clade. <span class="SmallCap">FIGURE 25.11b</span> illustrates how the data in our table of homologies translate into a cladogram.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig25-11','../fig/big/fig25-11.jpg','fig25-11');"><img border="0" src="../fig/thm/fig25-11.jpg" alt="fig25-11" name="myfig25-11"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 25-11.</B>&nbsp;<span><span class="Emphasis2">Constructing a cladogram.</span></span></td></tr></table><p class="Header3">Summary of Cladistic Analysis</p><a name="para13"></a><p class="BlockText">A cladogram represents the chronological sequence of branching during the evolutionary history of a set of organisms. Systematists infer this branching sequence by analyzing homologies, identifying the shared derived characters unique to each clade. In <span class="SmallCap">FIGURE 25.11b</span>, for example, we can infer from cladistic analysis that the turtle-leopard clade includes a common ancestor that is more recent than the ancestor of the salamander-turtle-leopard clade.</p><a name="para14"></a><p class="BlockText">Do not confuse this chronology of phylogenetic branching with the vintages of the organisms we are comparing. For example, the cladogram in <span class="SmallCap">FIGURE 25.11b</span> does <span class="Emphasis1">not</span> indicate that all turtles evolved more recently than all salamanders, but only that their common ancestor preceded the ancestor of the turtle-leopard clade. A particular species of turtle may be a very old member of that clade, while the salamander species used in the cladistic analysis may be a relatively recent member of an older clade. This is analogous to an early building in a relatively new city being older than a new building in a more ancient city. Note also that the chronology represented in a cladogram is relative (earlier versus later) rather than absolute (so many millions of years ago). Many of the evolutionary trees you will see in the next unit of chapters and elsewhere in this book are based on a combination of cladistic analysis and the use of fossil evidence and other data to place the phylogenetic branch points in the context of geologic time. (See <span class="SmallCap">FIGURE 29.1</span> for an example.)</p><a name="para15"></a><p class="BlockText">Systematists can use the cladograms they construct to place species in the taxonomic hierarchy of groups within groups, reflecting the nesting of clades within more comprehensive clades. <span class="SmallCap">FIGURE 25.12</span>, page 498, applies this principle to a group of mammals.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig25-12','../fig/big/fig25-12.jpg','fig25-12');"><img border="0" src="../fig/thm/fig25-12.jpg" alt="fig25-12" name="myfig25-12"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 25-12.</B>&nbsp;<span><span class="Emphasis2">Cladistics and taxonomy.</span> We can use the pattern of clades nested within clades to order species into a hierarchy of higher taxa. In this cladogram of mammals, a reptile (turtle) serves as the outgroup. The crossbars preceding branch points along the cladogram are labeled with examples of derived characters that evolved in the ancestor of each clade.</span></td></tr></table><a name="para16"></a><p class="BlockText">Some systematists now argue that the hierarchical system of classification is antiquated because it is so disruptive to rearrange taxa when a cladogram is revised based on new evidence. These systematists propose replacing the Linnaean system with a strictly cladistic classification called <span class="Emphasis2">phylocode.</span> Advocates of this alternative taxonomy simply name clades without the hierarchical tags, such as class, order, and family. For example, two clades we can extract from <span class="SmallCap">FIGURE 25.11b</span> might be named clade Vertebrata and clade Amniota (the vertebrates with amniotic, or shelled, eggs). So far, most biologists still prefer a hierarchical system of taxonomic levels as a more useful way of organizing the diversity of life.</p><a name="para17"></a><p class="BlockText">Each diagram of phylogeny, be it a cladogram or an evolutionary tree that incorporates a time scale, represents a hypothesis or a set of hypotheses about how the organisms in the tree are related. New evidence can compel systematists to revise their trees. And such reassessment has accelerated with the application of molecular methods for comparing species and tracing phylogeny.</p></td></tr></table></body></html>
