<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Molecular systematics is leading to a phylogenetic classification of prokaryotes</p><a name="para01"></a><p class="BlockText">A classification of prokaryotic life that reflects evolutionary history was impossible until the development of molecular systematics. Neither the fossil record nor comparative morphology are of much help in working out details of prokaryotic phylogeny. The major lineages diverged during the very early history of life and left a limited fossil record. And because prokaryotes are so structurally simple compared to macroscopic, eukaryotic organisms, comparing the structure of modern prokaryotic species does not turn up much in the way of derived characters that are useful in cladistic analysis (see Chapter 25).</p><a name="para02"></a><p class="BlockText">The breakthrough came when Carl Woese and his colleagues, mentioned earlier, began to cluster contemporary prokaryotes into taxonomic groups based on comparisons of nucleic acid sequences. The molecular material that turned out to be particularly useful is the RNA found in the smaller of the two subunits of ribosomes. Because all cells have ribosomes, Woese reasoned that the gene for the small-subunit ribosomal RNA (SSU-rRNA) has a history extending from the very first prokaryotes to the entire diversity of modern life. Each group of organisms has <span class="Emphasis2">signature sequences,</span> regions of its SSU-rRNA that have unique nucleotide sequences acquired by an accumulation of mutations in the ancestor of that taxonomic group. Thus, Woese found a way to define the very deepest branchings in the tree of life. <span class="SmallCap">FIGURE 27.13</span> (p. 536) diagrams the phylogeny of some of the major groups of prokaryotes as sorted by molecular systematics.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig27-13','../fig/big/fig27-13.jpg','fig27-13');"><img border="0" src="../fig/thm/fig27-13.jpg" alt="fig27-13" name="myfig27-13"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 27-13.</B>&nbsp;<span><span class="Emphasis2">Some major groups of prokaryotes.</span> This tentative phylogeny, like all such trees, is a hypothesis about evolutionary history. This simplified tree--consisting only of those prokaryotic groups that are discussed in this chapter--is based on molecular systematics, particularly comparisons of SSU-rRNA signature sequences. The groups shown here might be considered kingdoms within domains Bacteria and Archaea. However, many systematists prefer to wait for more data before assigning the groups to ranked taxonomic categories, such as kingdoms and phyla. This tree includes a "universal ancestor" common to all life, but we’ll see evidence in Chapter 28 that the deepest branchings of life may have emerged <span class="Emphasis1">not</span> from a single ancestor but from an ancestral community of gene-swapping prokaryotes.</span></td></tr></table><a name="para03"></a><p class="BlockText">Before molecular systematics, prokaryotic taxonomy was based on such phenotypic characters as nutritional mode and gram-positive versus gram-negative staining behavior (see <span class="SmallCap">FIGURE 27.5</span>). Such criteria remain useful for the identification of pathogenic bacteria in the clinical labs of hospitals. The telltale signs of a particular bacterial species cultured from a patient’s blood or other body fluid may be its shape as viewed with a microscope, its response to the Gram stain, whether or not the pathogen is motile, and what nutrients are required to grow the bacteria on a culture medium. For the most part, however, these clinically useful phenotypes are a poor guide to phylogeny. We have seen, for example, how the various nutritional modes are scattered around the phylogenetic tree of prokaryotes. However, a few of the traditional phenotype-based groups <span class="Emphasis1">do</span> persist in a phylogenetic classification based on molecular methods. For instance, note in <span class="SmallCap">FIGURE 27.13</span> that the cyanobacteria and the spirochetes cluster into two distinct lineages in the prokaryotic phylogeny. And although the gram-positive bacteria are very diverse, they all cluster into a single major prokaryotic branch. However, the gram-negative bacteria are scattered throughout several lineages; there are even some species that are gram-negative in staining behavior but are now included in the taxon called "gram-<span class="Emphasis1">positive</span>," so named because <span class="Emphasis1">most</span> of its members <span class="Emphasis1">are</span> gram-positive.</p><a name="para04"></a><p class="BlockText">Researchers have now sequenced the complete ge nomes of several prokaryotes, producing an enormous database that has supported most of the taxonomic conclusions based on SSU-rRNA comparison but has also produced some surprises. For example, in Chapter 28 we’ll discuss how rampant gene swapping within early communities of prokaryotes left a record in the genomes of all modern organisms, including humans.</p></td></tr></table></body></html>
