<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Restriction fragment analysis detects DNA differences that affect restriction sites</p><a name="para01"></a><p class="BlockText">Restriction fragment analysis indirectly detects certain differences in the nucleotide sequences of DNA molecules. In this method, we use gel electrophoresis to sort by size the DNA fragments that result from treating a long DNA molecule with a restriction enzyme. Just viewing the stained pattern of bands on the gel can provide scientifically useful information. When the mixture of restriction fragments from a particular DNA molecule undergoes electrophoresis, it yields a band pattern characteristic of the starting molecule and the restriction enzyme used. In fact, the relatively small DNA molecules of viruses and plasmids can be identified simply by their restriction fragment patterns. (Larger DNA molecules, such as those of eukaryotic chromosomes, yield too many fragments to appear as distinct bands.) Because DNA can be recovered undamaged from gels, the procedure also provides a way to prepare pure samples of individual fragments.</p><a name="para02"></a><p class="BlockText">Let’s use restriction fragment analysis to compare two different DNA molecules representing, for example, different alleles of a gene. We start by digesting each DNA sample with the same restriction enzyme. Because the two alleles must differ slightly in DNA sequence, they may differ in one or more restriction sites. Thus, the collections of restriction fragments from the two alleles may produce different band patterns in gel electrophoresis. <span class="SmallCap">FIGURE 20.9</span> (p. 384) shows how restriction fragment analysis by electrophoresis can distinguish between two alleles of a gene that differ in sequence by only one base pair if a restriction site is affected.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-9','../fig/big/fig20-9.jpg','fig20-9');"><img border="0" src="../fig/thm/fig20-9.jpg" alt="fig20-9" name="myfig20-9"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-9.</B>&nbsp;<span><span class="Emphasis2">Using restriction fragment patterns to distinguish DNA from different alleles.</span></span></td></tr></table><a name="para03"></a><p class="BlockText">The starting materials in <span class="SmallCap">FIGURE 20.9</span> are samples of cloned and purified genes. However, by combining gel electrophoresis with nucleic acid hybridization, we can make the same comparison without preliminary gene cloning. We can start with DNA of the whole genome. Although electrophoresis will yield too many bands to distinguish individually, we can use nucleic acid hybridization with a specific probe to label discrete bands that derive from our gene of interest.</p><a name="para04"></a><p class="BlockText">The principle is the same as for the nucleic acid hybridization in <span class="SmallCap">FIGURE 20.4</span>: A radioactive single-stranded nucleic acid probe selectively hydrogen-bonds with a targeted, complementary DNA sequence, which is then detected by autoradiography. Now, however, we use a probe in combination with an additional technique called Southern hybridization or <span class="Emphasis2">Southern blotting.</span><span class="SmallCap">FIGURE 20.10</span> outlines the entire procedure and demonstrates how it can be used to compare DNA samples from three individuals. This method goes beyond <span class="SmallCap">FIGURE 20.4</span> by revealing not only whether a particular sequence is present in a sample of DNA but also the restriction fragments that contain the sequence.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-10','../fig/big/fig20-10.jpg','fig20-10');"><img border="0" src="../fig/thm/fig20-10.jpg" alt="fig20-10" name="myfig20-10"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-10.</B>&nbsp;<span><span class="Emphasis2">Restriction fragment analysis by Southern blotting.</span> This procedure enables researchers to detect and analyze particular DNA sequences. The basis for detecting specific sequences is nucleic acid hybridization. The results can show not only whether certain sequences exist in different samples but also the number of such sequences within a genome and the size of the restriction fragments that contain them. In this way, it is possible to compare the DNA of different individuals or even species. Because of the selective power of nucleic acid hybridization, the starting material for analysis can be the entire genome of an organism. This huge length of DNA will produce so many restriction fragments that if all were made visible (with a dye, for example), they would appear as a smear in gel electrophoresis, rather than as discrete bands. Instead, only the DNA bands of interest are made visible by using a labeled probe. Before hybridization, the DNA being tested is transferred by capillary action (blotting) from the gel to a solid support, a sheet of nitrocellulose or nylon paper. The entire hybridization procedure is known as Southern blotting, after E.M. Southern, who developed it in 1975. The sensitivity of the hybridization can be adjusted either to detect only those sequences that are perfectly complementary to the probe or to detect similar (homologous) sequences as well.</span></td></tr></table><a name="para05"></a><p class="BlockText">Southern blotting proved invaluable when biologists turned their attention to <span class="Emphasis1">noncoding</span> DNA, which comprises most of the DNA of animal and plant genomes. Are there differences in noncoding DNA sequences analogous to the differences in alleles of genes? Using noncoding DNA in procedures like that of <span class="SmallCap">FIGURE 20.9</span>, researchers were excited to discover many differences in band patterns. Differences in DNA sequence on homologous chromosomes that can result in different restriction fragment patterns turn out to be scattered abundantly throughout genomes, including the human genome. Such differences have been named <span class="Emphasis2">restriction fragment length polymorphisms</span> (<span class="Emphasis2">RFLPs,</span> pronounced "Riflips"). This type of difference is conceptually the same as a difference in coding sequence; it, too, can serve as a genetic marker for a particular location (locus) in the genome. A given RFLP marker frequently occurs in numerous variants in a population. (The word <span class="Emphasis1">polymorphisms</span> comes from the Greek for "many forms.")</p><a name="para06"></a><p class="BlockText">RFLPs are detected and analyzed by Southern blotting. The example shown in <span class="SmallCap">FIGURE 20.10</span> could as easily represent the detection of a RFLP in noncoding DNA as a difference in the coding sequences of two alleles. Because of the sensitivity of DNA hybridization, the entire genome can be used as the DNA starting material. (Samples of human DNA are typically obtained from white blood cells.) The result of the procedure shown in <span class="SmallCap">FIGURE 20.10</span> indicates that individuals I and II carry the same version of the marker (RFLP or gene), but individual III carries a different version.</p><a name="para07"></a><p class="BlockText">Because RFLP markers are inherited in a Mendelian fashion, they can serve as genetic markers for making linkage maps. The geneticist uses the same reasoning you saw applied in <span class="SmallCap">FIGURE 15.6</span>: The frequency with which two RFLP markers--or a RFLP marker and a certain allele for a gene--are inherited together is a measure of the closeness of the two loci on a chromosome. The discovery of RFLPs greatly increased the number of markers available for mapping the human genome. No longer were geneticists limited to genetic variations that lead to obvious phenotypic differences (such as genetic diseases) or even to differences in protein products.</p></td></tr></table></body></html>
