<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Genetic recombination produces new bacterial strains</p><a name="para01"></a><p class="BlockText">Natural selection depends on heritable variation among the individuals of a population (see Chapter 1). In addition to mutations, genetic recombination generates diversity within bacterial populations. We will define recombination here simply as the combining of DNA from two individuals into the genome of a single individual.</p><a name="para02"></a><p class="BlockText">How can we detect genetic recombination in bacteria? Consider two mutant <span class="Emphasis1">E. coli</span> strains (genetic varieties), each unable to synthesize one of its required amino acids. Wild-type <span class="Emphasis1">E. coli</span> can grow on a minimal medium containing only glucose, as a source of organic carbon, and salts. The mutant strains cannot grow on this culture medium of minimal nutrients because one of them cannot synthesize tryptophan and the other cannot synthesize arginine (<span class="SmallCap">FIGURE 18.12</span>).</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-12','../fig/big/fig18-12.jpg','fig18-12');"><img border="0" src="../fig/thm/fig18-12.jpg" alt="fig18-12" name="myfig18-12"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-12.</B>&nbsp;<span><span class="Emphasis2">Detecting genetic recombination in bacteria.</span> Each of the two mutant strains of <span class="Emphasis1">E. coli</span> in this experiment is unable to synthesize a particular amino acid, either arginine or tryptophan. These mutant bacteria cannot reproduce on minimal medium (MM), which is simply a solution of glucose and salts. But after bacteria of the two mutant strains are incubated together, some cells from the mixed culture are able to grow into colonies on a plate containing minimal medium solidified with agar. Those cells are recombinant bacteria: <span class="Emphasis1">arg</span><SUP>+</SUP><span class="Emphasis1">trp</span><SUP>+</SUP> cells produced by gene transfer between cells of the two mutant types. (You will learn about the mechanism in <span class="SmallCap">FIGURE 18.15c</span> and d.)</span></td></tr></table><a name="para03"></a><p class="BlockText">Let’s suppose we mix bacteria from the two strains together in a liquid medium and allow them to incubate for an hour or so. We then spread a small sample of this culture on solid (agar-thickened) minimal medium in a petri dish and incubate the dish overnight. The next morning we observe numerous colonies of bacteria on the minimal medium. Each of these colonies must have started with a cell capable of making <span class="Emphasis1">both</span> tryptophan <span class="Emphasis1">and</span> arginine, but their number far exceeds what can be accounted for by mutation. Most of the cells that can synthesize both amino acids must have acquired one or more genes from the other strain. This is evidence that genetic recombination has occurred.</p><a name="para04"></a><p class="BlockText">Bacteria are different from eukaryotes in the ways DNA from two individuals can come together in one cell. In eukaryotes, the sexual processes of meiosis and fertilization combine DNA from two individuals in a single zygote (see Chapter 13). But sex, as defined in eukaryotes, is absent in prokaryotes: Meiosis and fertilization do not occur. Instead, other processes bring together bacterial DNA from different individuals. These processes are transformation, transduction, and conjugation.</p><p class="Header3">Transformation</p><a name="para05"></a><p class="BlockText">In the context of bacterial genetics, <span class="Emphasis2">transformation</span> is the alteration of a bacterial cell’s genotype by the uptake of naked, foreign DNA from the surrounding environment. For example, we saw in Chapter 16 that harmless <span class="Emphasis1">Streptococcus pneumoniae</span> bacteria could be transformed to pneumonia-causing cells by the uptake of naked DNA from a medium containing dead, broken-open cells of the pathogenic strain. This transformation occurs when a live nonpathogenic cell takes up a piece of DNA that happens to include the allele for pathogenicity (the gene for a cell coat that protects the bacterium from a host’s immune system). The foreign allele is then incorporated into the bacterial chromosome, replacing the native allele (for the "coatless" condition, in this case). The process is genetic recombination--an exchange of DNA segments by crossing over. The cell is now a recombinant: Its chromosome contains DNA derived from two different cells.</p><a name="para06"></a><p class="BlockText">For many years after transformation was discovered in laboratory cultures, most biologists believed the process to be too rare and haphazard to play an important role in natural bacterial populations. But researchers have since learned that many bacterial species possess on their surfaces proteins that are specialized for the uptake of naked DNA from the surrounding solution. These proteins specifically recognize and transport only DNA from closely related species of bacteria. Not all bacteria have such membrane proteins. For instance, <span class="Emphasis1">E. coli</span> does not seem to have any specialized mechanism for the uptake of foreign DNA. However, placing <span class="Emphasis1">E. coli</span> in a culture medium containing a relatively high concentration of calcium ions will artificially stimulate the cells to take up small pieces of DNA. In biotechnology, this technique is applied to introduce foreign genes into <span class="Emphasis1">E. coli</span>--genes coding for valuable proteins, such as human insulin and growth hormone.</p><p class="Header3">Transduction</p><a name="para07"></a><p class="BlockText">In the DNA transfer process known as <span class="Emphasis2">transduction,</span> phages (the viruses that infect bacteria) carry bacterial genes from one host cell to another. There are two forms of transduction: generalized transduction and specialized transduction. Both result from aberrations in phage reproductive cycles.</p><a name="para08"></a><p class="BlockText">First let’s consider generalized transduction, shown on the left side of <span class="SmallCap">FIGURE 18.13</span>. Recall that near the end of a phage’s lytic cycle, viral nucleic acid molecules are packaged within capsids, and the completed phages are released when the host cell lyses. Occasionally, a small piece of the host cell’s degraded DNA is accidentally packaged within a phage capsid in place of the phage genome. Such a virus is defective because it lacks its own genetic material. However, after its release from the lysed host, the phage can attach to another bacterium and inject the piece of bacterial DNA acquired from the first cell. Some of this DNA can subsequently replace the homologous region of the second cell’s chromosome. The cell’s chromosome now has a combination of DNA derived from two cells; genetic recombination has occurred. This type of transduction is called <span class="Emphasis2">generalized transduction</span> because the phage transfers bacterial genes at random.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-13','../fig/big/fig18-13.jpg','fig18-13');"><img border="0" src="../fig/thm/fig18-13.jpg" alt="fig18-13" name="myfig18-13"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-13.</B>&nbsp;<span><span class="Emphasis2">Transduction.</span> Phages occasionally carry bacterial genes from one cell to another. In generalized transduction (left), random pieces of the host chromosome are packaged within a phage capsid. In specialized transduction (right), a prophage exits the chromosome in such a way that it carries adjacent bacterial genes along with it. In both types of transduction, the transferred DNA may recombine with the genome of the new host cell.</span></td></tr></table><a name="para09"></a><p class="BlockText">Let’s contrast this process with specialized transduction, shown on the right side of <span class="SmallCap">FIGURE 18.13</span>. This form of transduction requires infection by a temperate phage. Recall that in the lysogenic cycle, the genome of a temperate phage integrates as a prophage into the host bacterium’s chromosome, usually at a specific site. Later, when the phage genome is excised from the chromosome, it sometimes takes with it a small region of the bacterial DNA that was adjacent to the prophage. When such a virus carrying bacterial DNA infects another host cell, the bacterial genes are injected along with the phage’s genome. <span class="Emphasis2">Specialized transduction</span> transfers only certain genes, those near the prophage site on the bacterial chromosome.</p><p class="Header3">Conjugation and Plasmids</p><a name="para10"></a><p class="BlockText"><span class="Emphasis2">Conjugation</span> is the direct transfer of genetic material between two bacterial cells that are temporarily joined. This process, the bacterial version of sex, has been studied most extensively in <span class="Emphasis1">E. coli</span>. The DNA transfer is one-way: one cell donating DNA, and its "mate" receiving the genes. The DNA donor, referred to informally as the "male," uses appendages called sex pili to attach to the DNA recipient, the "female" (<span class="SmallCap">FIGURE 18.14</span>). A sex pilus acts like a grappling hook: After contacting a female cell, it retracts, pulling the two cells together. A temporary cytoplasmic bridge then forms between the two cells, providing an avenue for DNA transfer. In most cases, "maleness," the ability to form sex pili and donate DNA during conjugation, results from the presence of a special piece of DNA called an <span class="Emphasis2">F factor</span> (F for fertility). An F factor can exist either as a segment of DNA within the bacterial chromosome or as a plasmid. Before discussing the role of the F factor in conjugation, we should examine plasmids more generally.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-14','../fig/big/fig18-14.jpg','fig18-14');"><img border="0" src="../fig/thm/fig18-14.jpg" alt="fig18-14" name="myfig18-14"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-14.</B>&nbsp;<span><span class="Emphasis2">Bacterial mating.</span> The <span class="Emphasis1">E. coli</span>"male" (right) extends sex pili, one of which is attached to a "female" cell. The two cells will be drawn close together, allowing a cytoplasmic bridge to form between them. Through this tube, the "male" will transfer DNA to the "female." This mechanism of DNA transfer is called conjugation (colorized TEM).</span></td></tr></table><a name="para11"></a><p class="BlockText"><span class="Emphasis2">General Characteristics of Plasmids.</span> A <span class="Emphasis2">plasmid</span> is a small, circular, self-replicating DNA molecule separate from the bacterial chromosome. Certain plasmids, such as F plasmids, can undergo reversible incorporation into the cell’s chromosome. A genetic element that can exist either as a plasmid or as part of the bacterial chromosome is called an <span class="Emphasis2">episome.</span> In addition to some plasmids, temperate viruses, such as phage <span class="symbol">l</span>, also qualify as episomes. Recall that the genomes of these phages replicate independently during a lytic cycle and as an integral part of the bacterial chromosome during a lysogenic cycle. Of course, there are important differences between plasmids and viruses. Plasmids, unlike viruses, lack protein coats and do not normally exist outside the cell. And plasmids are generally beneficial to the bacterial cell, while viruses are parasites that usually harm their hosts.</p><a name="para12"></a><p class="BlockText">A plasmid has only a small number of genes, and these genes are not required for the survival and reproduction of the bacterium under normal conditions. However, the genes of plasmids can confer advantages on bacteria living in stressful environments. For example, the F plasmid facilitates genetic recombination, which may be advantageous in a changing environment that no longer favors existing strains in a bacterial population.</p><a name="para13"></a><p class="BlockText"><span class="Emphasis2">The F Plasmid and Conjugation.</span> The F factor and its plasmid form, the <span class="Emphasis2">F plasmid,</span> consist of about 25 genes, most required for the production of sex pili. Geneticists use the symbol F<SUP>+</SUP> to denote a cell that contains the F plasmid (a "male" cell). The F<SUP>+</SUP> condition is heritable: The F plasmid replicates in synchrony with the chromosomal DNA, and division of an F<SUP>+</SUP> cell usually gives rise to two offspring that are both F<SUP>+</SUP>. Cells lacking the F factor in either form are designated F<SUP>-</SUP>, and they function as DNA recipients ("females") during conjugation. The F<SUP>+</SUP> condition is "contagious" in the sense that an F<SUP>+</SUP> cell converts an F<SUP>-</SUP> cell to F<SUP>+</SUP> when the two cells conjugate. The original F<SUP>+</SUP> cell remains F<SUP>+</SUP> because the F plasmid replicates within the "male" cell, and only one copy of the plasmid is transferred to the "female" through the conjugation tube joining the cells (<span class="SmallCap">FIGURE 18.15a</span>, p. 344). In an F<SUP>+</SUP>&#160;X&#160; F<SUP>-</SUP> mating, only an F plasmid is transferred.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-15','../fig/big/fig18-15.jpg','fig18-15');"><img border="0" src="../fig/thm/fig18-15.jpg" alt="fig18-15" name="myfig18-15"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-15.</B>&nbsp;<span><span class="Emphasis2">Conjugation and recombination in <span class="Emphasis1">E. coli.</span></span> The DNA replication that accompanies transfer of an F plasmid or part of an Hfr bacterial chromosome is called <span class="Emphasis1">rolling-circle replication.</span></span></td></tr></table><a name="para14"></a><p class="BlockText">Under what circumstances are genes of the bacterial chromosome transferred during conjugation? This can occur when the donor cell’s F factor is integrated into the chromosome (<span class="SmallCap">FIGURE 18.15b</span>). A cell with the F factor built into its chromosome is called an Hfr cell (for high frequency of recombination). Like an F<SUP>+</SUP> cell, an Hfr cell functions as a male during conjugation: It initiates DNA replication at a point on the F factor DNA and from that point starts to transfer the DNA copy to its F<SUP>-</SUP> partner. But now, the leading end of the F factor is dragging a copy of chromosomal DNA along with it (<span class="SmallCap">FIGURE 18.15c</span>). Random movements of the bacteria almost always disrupt conjugation long before an entire copy of the Hfr chromosome can be passed to the F<SUP>-</SUP> cell. Temporarily, the recipient cell is a partial diploid, containing its own chromosome plus DNA copied from part of the donor’s chromosome. Then recombination can occur: If part of the newly acquired DNA aligns with the homologous region of the F<SUP>-</SUP> chromosome, segments of DNA can be exchanged (<span class="SmallCap">FIGURE 18.15d</span>). Binary fission of this cell gives rise to a colony of recombinant bacteria with genes derived from two different cells. (This is what happened in the experiment in <span class="SmallCap">FIGURE 18.12</span>, where one of the bacterial strains was an Hfr and the other an F<SUP>-</SUP>.)</p><a name="para15"></a><p class="BlockText"><span class="Emphasis2">R Plasmids and Antibiotic Resistance.</span> In the 1950s, Japanese physicians began to notice that some hospital patients suffering from bacterial dysentery, which produces severe diarrhea, did not respond to antibiotics that had generally been effective in treating this type of infection. Apparently, re-sis-tance to these antibiotics had evolved in certain strains of <span class="Emphasis1">Shigella,</span> the pathogen. Eventually, researchers began to identify the specific genes that confer antibiotic resistance in <span class="Emphasis1">Shigella</span> and other pathogenic bacteria. Some of these genes, for example, code for enzymes that specifically destroy certain antibiotics, such as tetracycline or ampicillin. The genes conferring resistance, it turns out, are carried by plasmids, now known as <span class="Emphasis2">R plasmids</span> (R for resistance).</p><a name="para16"></a><p class="BlockText">Exposure of a bacterial population to a specific antibiotic, whether in a laboratory culture or within a host organism, will kill antibiotic-sensitive bacteria but not those that happen to have R plasmids that counter that antibiotic. The theory of natural selection predicts that under these circumstances, the fraction of the bacterial population carrying genes for anti-biotic resistance will increase, and that is exactly what happens. The medical consequences are also predictable: Resistant strains of pathogens are becoming more common, making the treatment of certain bacterial infections more difficult. The problem is compounded by the fact that R plasmids, like F plasmids, have genes that encode sex pili and enable plasmid transfer from one bacterial cell to another by conjugation. Making the problem still worse, some R plasmids carry as many as ten genes for resistance to that many antibiotics. How do so many antibiotic resistance genes become part of a single plasmid? The answer involves another type of mobile genetic element, called a transposon, which we investigate next.</p><p class="Header3">Transposons</p><a name="para17"></a><p class="BlockText">A <span class="Emphasis2">transposon,</span> also called a transposable genetic element, is a piece of DNA that can move from one location to another in a cell’s genome. Unlike an episome or prophage, transposons never exist independently. Instead, transposon movement (transposition) occurs as a type of recombination between the transposon and another DNA site--a target site--that comes in contact with the transposon. In a bacterial cell, a transposon may move within the chromosome, from a plasmid to the chromosome (or vice versa), or from one plasmid to another. Transposons bring multiple genes for antibiotic resistance into a single R plasmid by moving the genes to that location from different plasmids.</p><a name="para18"></a><p class="BlockText">Transposons are sometimes called "jumping genes," but the phrase is slightly misleading. Some transposons <span class="Emphasis1">do</span> jump from one genomic location to another, in what is called cut-and-paste transposition. However, in another type of transposition, called replicative transposition, the transposon replicates at its original site, and a <span class="Emphasis1">copy</span> inserts elsewhere; that is, the transposon is added at some new site without being lost from the old site.</p><a name="para19"></a><p class="BlockText">Although transposons vary in their selectivity for target sites, most can move to many alternative locations in the DNA. This ability to scatter certain genes throughout the genome makes transposition fundamentally different from other mechanisms of genetic shuffling. The genetic recombination that occurs in bacterial transformation, generalized transduction, and conjugation (and during meiosis in eukaryotes as well) depends on base pairing between homologous regions of DNA, regions of identical or very similar base sequence. In contrast, the insertion of a transposon in a new site does not depend on complementary base sequences. A transposon can move genes to a site where genes of that sort have never before existed.</p><a name="para20"></a><p class="BlockText"><span class="Emphasis2">Insertion Sequences.</span> The simplest bacterial transposons are <span class="Emphasis2">insertion sequences.</span> They consist of only the DNA necessary for the act of transposition. The one gene found in an insertion sequence codes for a transposase, an enzyme that catalyzes movement of the transposon from one location to another within the genome. The transposase gene is bracketed by a pair of DNA sequences called <span class="Emphasis1">inverted repeats,</span> noncoding sequences about 20 to 40 nucleotides long. Notice in <span class="SmallCap">FIGURE 18.16</span> that each base sequence is repeated in reverse along the opposite DNA strand of the inverted repeat at the other end of the transposon. Transposase recognizes these inverted repeats as the boundaries of the transposon. During transposition, molecules of the enzyme bind to the inverted repeats and to a target site elsewhere in the genome and catalyze the DNA cutting and resealing required for transposition. Other enzymes also participate in transposition. For example, DNA polymerase helps create identical regions of DNA, called <span class="Emphasis1">direct repeats,</span> that flank a transposon in its new site (<span class="SmallCap">FIGURE 18.17</span>, p. 346).</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-16','../fig/big/fig18-16.jpg','fig18-16');"><img border="0" src="../fig/thm/fig18-16.jpg" alt="fig18-16" name="myfig18-16"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-16.</B>&nbsp;<span><span class="Emphasis2">Insertion sequences, the simplest transposons.</span> The one gene of an insertion sequence codes for transposase, which catalyzes the transposon’s movement. The inverted repeats, about 20 to 40 nucleotide pairs long, are backward, upside-down versions of each other. In transposition, transposase molecules bind to the inverted repeats and catalyze the cutting and resealing of DNA required for insertion of the transposon at a target site. This diagram and the ones that follow are not to scale.</span></td></tr></table><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-17','../fig/big/fig18-17.jpg','fig18-17');"><img border="0" src="../fig/thm/fig18-17.jpg" alt="fig18-17" name="myfig18-17"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-17.</B>&nbsp;<span><span class="Emphasis2">Insertion of a transposon and creation of direct repeats.</span>&nbsp;
			&nbsp;<img align="middle" src="../img/sym/disk1.gif">&nbsp;
		 First, the transposase enzyme makes staggered cuts (red arrows) in the two DNA strands at a target site, leaving short segments of unpaired DNA as shown. Meanwhile, the transposon is cut out or copied at its initial site. 
			&nbsp;<img align="middle" src="../img/sym/disk2.gif">&nbsp;
		 The transposon is then joined to the single-stranded ends at the target site. Presumably, the transposase holds all the components together during this process. 
			&nbsp;<img align="middle" src="../img/sym/disk3.gif">&nbsp;
		 Finally, the gaps in the DNA strands are filled in by DNA polymerase and sealed by ligase. This results in <span class="Emphasis1">direct repeats,</span> identical segments of DNA on either side of the transposon. (Distances along the DNA are not to scale).</span></td></tr></table><a name="para21"></a><p class="BlockText">Insertion sequences cause mutations when they happen to land within the coding sequence of a gene or within a DNA region that regulates gene expression. But notice that this mechanism of mutation is intrinsic to the cell, in contrast to mutagenesis by extrinsic factors such as environmental radiation and chemicals. Insertion sequences account for about 1.5%  of the <span class="Emphasis1">E. coli</span> genome. However, mutation of a given gene by transposition occurs only rarely--about once in every 10 million generations. This is about the same as the spontaneous mutation rate due to other factors.</p><a name="para22"></a><p class="BlockText"><span class="Emphasis2">Composite Transposons.</span> Transposons longer and more complex than insertion sequences also move about in the bacterial genome. In addition to the DNA required for transposition, these <span class="Emphasis1">composite transposons</span> (also called complex transposons) include extra genes that go along for the ride, such as genes for antibiotic resistance. The extra genes are sandwiched between two insertion sequences (<span class="SmallCap">FIGURE 18.18</span>). It is as though two insertion sequences happened to land relatively close together in the genome and now travel together, along with all the DNA between them, as a single transposon. As is characteristic of transposons, there is an inverted repeat and a direct repeat at each end. (The direct repeat is not considered part of the transposon, however.)</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-18','../fig/big/fig18-18.jpg','fig18-18');"><img border="0" src="../fig/thm/fig18-18.jpg" alt="fig18-18" name="myfig18-18"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-18.</B>&nbsp;<span><span class="Emphasis2">Anatomy of a composite transposon.</span> A composite transposon consists of one or more genes located between twin insertion sequences. The transposon here has a gene for resistance to an antibiotic, which is carried along as part of the transposon when the transposon is inserted at a new site in the genome.</span></td></tr></table><a name="para23"></a><p class="BlockText">In contrast to insertion sequences, which are not known to benefit bacteria in any specific way, composite transposons may help bacteria adapt to new environments. We mentioned earlier that a single R plasmid can carry several genes for resistance to different antibiotics. This is explained by composite transposons, which can add a gene for antibiotic resistance to a plasmid already carrying genes for resistance to other antibiotics. The transmission of this composite plasmid to other bacterial cells by cell division or conjugation can then spread resistance to a variety of antibiotics throughout a bacterial population. In an antibiotic-rich environment, natural selection favors bacterial clones that have built up composite R plasmids through a series of transpositions.</p><a name="para24"></a><p class="BlockText">Transposable genetic elements are not unique to bacteria, but are important components of eukaryotic genomes as well. In fact, the first evidence for such wandering DNA segments came from the American geneticist Barbara McClintock’s breeding experiments with Indian corn (maize) in the 1940s and 1950s. McClintock identified changes in the color of corn kernels that made sense only if she postulated the existence of mobile genetic elements capable of moving from other locations in the genome to the genes for kernel color. She called these mobile elements "controlling -elements" because they seemed to insert next to the genes responsible for kernel color, either activating or inactivating those genes. McClintock’s discovery received little attention until transposons were discovered in bacteria many years later and microbial geneticists learned more about the molecular basis of transposition. In 1983, more than 30 years after she discovered transposable genetic elements, Barbara McClintock was awarded a Nobel Prize, at age 81. McClintock continued her experiments at Cold Spring Harbor Laboratory in New York until her death in 1992.</p><a name="para25"></a><p class="BlockText">You will learn more about transposons in eukaryotes in Chapter 19. We end this Chapter by examining how bacterial genes are regulated in different environments.</p></td></tr></table></body></html>
