<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Entire genomes can be mapped at the DNA level</p><a name="para01"></a><p class="BlockText">As early as 1980, molecular biologist David Botstein and colleagues proposed that the DNA variations reflected in RFLPs could serve as the basis for an extremely detailed map of the entire human genome. Since then, researchers have in fact used such markers for mapping the genomes of a number of organisms. For some organisms, they have already succeeded in bringing the map to the ultimate level of detail: the entire sequence of nucleotides in the DNA. Along the way, they have taken advantage of all the tools and techniques discussed in this chapter--restriction enzymes, DNA cloning, gel electrophoresis, labeled probes, and so forth.</p><a name="para02"></a><p class="BlockText">The most ambitious research project made possible by DNA technology has been the <span class="Emphasis2">Human Genome Project,</span> officially begun in 1990. This is an effort to map the entire human genome, ultimately by determining the complete nucleotide sequence of the DNA of each human chromosome (the 22 autosomes and the X and Y sex chromosomes). As organized by an international, publicly funded consortium of 20 groups of researchers at universities and research institutes, the project has proceeded through three stages that focus on the DNA more and more closely: genetic (or linkage) mapping, physical mapping, and DNA sequencing. We’ll describe these stages shortly.</p><a name="para03"></a><p class="BlockText">In addition to mapping human DNA, the Human Genome Project includes mapping the genomes of other species important in biological research, including <span class="Emphasis1">E. coli</span> and other prokaryotes, <span class="Emphasis1">Saccharomyces cerevisiae</span> (yeast), <span class="Emphasis1">Caenorhabditis elegans</span> (nematode), <span class="Emphasis1">Drosophila melanogaster</span> (fruit fly), and <span class="Emphasis1">Mus musculus</span> (mouse). These genomes are of great interest in their own right and are also providing important insights of general biological significance, as we’ll discuss later. In addition, the early work on these genomes was useful for developing the strategies, methods, and new technologies necessary for deciphering the human genome, which is much larger. The technological power for this daunting task has come in large part from advances in automation and from utilization of electronic technology, including computer software.</p><p class="Header3">Genetic (Linkage) Mapping</p><a name="para04"></a><p class="BlockText">In mapping a large genome, the first stage is to construct a linkage map of several thousand genetic markers spaced evenly throughout the chromosomes. As you learned in Chapter 15, the order of the markers on such a map and the relative distances between them are based on recombination frequencies. The markers can be genes (as in Chapter 15) or any other identifiable sequences in the DNA, such as RFLPs or the short repetitive sequences called microsatellites (see TABLE 19.1). Relying primarily on microsatellites, which are abundant in the human genome and have various "alleles" differing in length, researchers completed in a few years a human genetic map with some 5,000 markers. Such a map enables them to locate other markers, including genes, by testing for genetic linkage to the known markers. It is also valuable as a framework for organizing more detailed maps of particular regions.</p><p class="Header3">Physical Mapping: Ordering DNA Fragments</p><a name="para05"></a><p class="BlockText">In a physical map, the distances between markers are expressed in some physical measure, usually the number of nucleotides along the DNA. For whole-genome mapping, a physical map is made by cutting the DNA of each chromosome into a number of identifiable restriction fragments and then determining the original order of the fragments in the chromosomal DNA. The key is to make fragments that overlap and then use probes or automated nucleotide sequencing of the ends to find the overlaps. A method called <span class="Emphasis2">chromosome walking,</span> which uses probes, has been very useful (<span class="SmallCap">FIGURE 20.11</span>).</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-11','../fig/big/fig20-11.jpg','fig20-11');"><img border="0" src="../fig/thm/fig20-11.jpg" alt="fig20-11" name="myfig20-11"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-11.</B>&nbsp;<span><span class="Emphasis2">Chromosome walking.</span> The researcher starts with a known gene or other DNA segment--one that has already been cloned, mapped, and sequenced--&gt;and "walks" along the chromosomal DNA from that locus, producing a map of overlapping restriction fragments. To prepare the fragments, the scientist cuts two samples of chromosomal DNA with different restriction enzymes and then clones both sets of fragments to form two libraries. This and other methods of ordering DNA fragments involve determining the nucleotide sequences of short, unique segments within each fragment. The resulting physical map is thus a linear array of fragments, each identifiable by two or more sequenced regions. All the DNA in this diagram is shown as single strands.</span></td></tr></table><a name="para06"></a><p class="BlockText">Supplies of the DNA fragments used for physical mapping are prepared by cloning. In working with large genomes, researchers carry out several rounds of DNA cutting, cloning, and physical mapping. The first cloning vector is often a yeast artificial chromosome (YAC), which can carry inserted fragments a million base pairs long, or a <span class="Emphasis2">bacterial artificial chromosome (BAC),</span> an artificial version of a bacterial chromosome that can carry inserts of 100,000 to 500,000 base pairs. The scientists determine the order of these long fragments (by chromosome walking, for instance) and then cut each fragment into smaller pieces, which are cloned and ordered in turn. The final sets of fragments, cloned in plasmids or phage, are only about 1,000 base pairs long, short enough to be sequenced easily.</p><p class="Header3">DNA Sequencing</p><a name="para07"></a><p class="BlockText">The complete nucleotide sequence of a genome is the ultimate map. To repeat a key point, it is the technology for cloning fragments of DNA that has made possible the sequencing of the very long DNA molecules of genomes. Starting with a pure preparation of many copies of a relatively short DNA fragment, the nucleotide sequence of the fragment can be determined by a sequencing machine (see the photograph opening this chapter). The usual sequencing technique, described in <span class="SmallCap">FIGURE 20.12</span> (p. 388), combines DNA labeling, DNA synthesis involving special chain-terminating nucleotides, and high-resolution gel electrophoresis. But even with automation, the sequencing of all 3.2 billion nucleotide pairs in a haploid set of human chromosomes is a formidable challenge. In fact, a major thrust of the Human Genome Project has been the development of technology for faster sequencing, along with more sophisticated computer software for analyzing and assembling the partial sequences.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-12','../fig/big/fig20-12.jpg','fig20-12');"><img border="0" src="../fig/thm/fig20-12.jpg" alt="fig20-12" name="myfig20-12"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-12.</B>&nbsp;<span><span class="Emphasis2">Sequencing of DNA by the Sanger method.</span> Developed by British scientist Frederick Sanger, this method for determining the nucleotide sequence of DNA molecules involves synthesizing <span class="Emphasis1">in vitro</span> DNA strands complementary to one of the strands of the DNA being sequenced. The method is based on the random incorporation of a modified nucleotide, a dideoxyribonucleotide, which terminates a growing DNA chain because it lacks a 3<B>&#39;</B>--OH for attachment of the next nucleotide (see <span class="SmallCap">FIGURE 16.11</span>). DNA strands are synthesized that reflect all possible positions of the dideoxynucleotides and thus ultimately the entire DNA sequence. The molecules that are sequenced are cloned restriction fragments, typically several hundred nucleotides long. Most of the work of DNA sequencing is now automated. Instead of labeling the fragments with radioactive primers, modern sequencing machines use fluorescent dyes to tag the dideoxyribonucleotides, one color for each of the four types of nucleotide. This approach allows the four reactions to be performed in a single tube; the dideoxy ends of the reaction product DNA strands can be distinguished by the color of their fluorescence (see chapter-opening photo).</span></td></tr></table><a name="para08"></a><p class="BlockText"><span class="SmallCap">FIGURE 20.13a</span> (p. 389) reviews the three-stage, hierarchical approach to genome mapping that we’ve discussed so far. In practice, the three stages have overlapped in a way that our simplified version does not portray, but they have remained the overarching strategy of the public consortium of researchers.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-13','../fig/big/fig20-13.jpg','fig20-13');"><img border="0" src="../fig/thm/fig20-13.jpg" alt="fig20-13" name="myfig20-13"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-13.</B>&nbsp;<span><span class="Emphasis2">Alternative strategies for sequencing an entire genome.</span> In both cases, the arrangement of DNA fragments in order depends on their having overlapping regions.</span></td></tr></table><p class="Header3">Alternative Approaches to Whole-Genome Sequencing</p><a name="para09"></a><p class="BlockText">In 1992, emboldened by advances in sequencing machines and computer technology, molecular biologist J. Craig Venter came up with an alternative approach to the sequencing of whole genomes. His idea was essentially to skip the genetic mapping and physical mapping stages and start directly with the sequencing of random DNA fragments. Powerful computer programs would then assemble the resulting very large number of overlapping short sequences into a single continuous one (<span class="SmallCap">FIGURE 20.13b</span>). Despite the skepticism of many other scientists, Venter left the public consortium to follow his vision. The worth of his approach was demonstrated in 1995, at least for prokaryotic genomes, when he and colleagues reported the first complete sequence of an organism, the bacterium <span class="Emphasis1">Hemophilus influenzae.</span> In May 1998, he set up a company, Celera Genomics, and promised a completed human sequence in three years. His whole-genome shotgun approach to sequencing was further validated in March 2000 by the sequence of <span class="Emphasis1">Drosophila melanogaster,</span> carried out in collaboration with academic scientists. As promised, in February 2001, Celera announced the sequencing of over 90%  of the human genome--simultaneously with a similar announcement by the public consortium.</p><a name="para10"></a><p class="BlockText">Representatives of the public consortium have pointed out that Celera made heavy use of their maps and sequence data, which are all--unlike Celera’s data--made immediately available for free use, and assert that the infrastructure established by their approach will greatly facilitate the finishing-up phase of the project. Venter, on the other hand, argues for the efficiency and economy of Celera’s methods; and indeed, the public consortium has made some use of them. What is clear is that both approaches are valuable and that the competition between the two groups has hastened progress. (See <span class="SmallCap">FIGURE 20.13</span> to review the two strategies.)</p><a name="para11"></a><p class="BlockText">As of mid-2001, the genomes of about 50 species have been completely (or almost completely) sequenced. Most are prokaryotes, including <span class="Emphasis1">E. coli,</span> a number of other bacteria (including ones of medical importance), and about 10 archaea. The yeast <span class="Emphasis1">Saccharomyces cerevisiae</span> was the first eukaryote to be completed, and the nematode <span class="Emphasis1">C. elegans,</span> a simple worm, the first multicellular organism. The plant <span class="Emphasis1">Arabidopsis thaliana,</span> another important research organism, has also been completed. Finally, the human genome is well on its way to completion, although there are still many gaps in the sequence. Because of the presence of repetitive DNA and for other poorly understood reasons, certain parts of the chromosomes of multicellular organisms resist detailed mapping by the usual methods. However, as sequences for more organisms are assembled, the job gets easier. For example, the sequencing of the mouse genome, which is about 85%  identical to the human genome, is being greatly aided by what is now known of the human sequence.</p><a name="para12"></a><p class="BlockText">On one level these sequences are simply dry lists of nucleotide bases--millions of A’s, T’s, C’s, and G’s in mind-numbing succession. But on another level, reached with the help of computer analysis, these sequences are already the basis of some exciting discoveries, which we discuss next.</p></td></tr></table></body></html>
