<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Plant development depends on cell signaling and transcriptional regulation</p><a name="para01"></a><p class="BlockText">The last common ancestor of plants and animals was probably a single-celled microbe living hundreds of millions of years ago, so the processes of development must have evolved independently in the two lineages of organisms. Plants evolved with rigid cell walls that make the movement of cells and tissue layers virtually impossible, ruling out a morphogenetic mechanism that is important in animals. Plant morphogenesis relies more heavily on differing planes of cell division and on selective cell enlargement. (You will learn about these processes in Chapter 35.)</p><a name="para02"></a><p class="BlockText">But despite the differences between plants and animals, there are some basic similarities in their molecular, cellular, and genetic mechanisms of development--legacies of their shared cellular origins. In particular, plant development, like that of animals, depends on cell signaling (induction) and transcriptional regulation.</p><a name="para03"></a><p class="BlockText">We are just beginning to understand the molecular basis of plant development in detail. Thanks to DNA technology, clues from animal research, and model organisms such as<span class="Emphasis1">Arabidopsis</span>, plant research is now progressing rapidly. The embryonic development of most plants occurs inside the seed and is relatively inaccessible to study (a mature seed already contains a fully formed embryo). However, other important aspects of plant development are observable throughout the plant’s life in its meristems, and in particular in the apical meristems at the tips of shoots. It is there that cell division, morphogenesis, and differentiation give rise to new organs, such as leaves or the petals of flowers. We’ll discuss two examples of research on floral meristems, the apical meristems that produce flowers.</p><p class="Header3">Cell Signaling in Flower Development</p><a name="para04"></a><p class="BlockText">Environmental signals, such as day length and temperature, trigger signal-transduction pathways that convert ordinary shoot meristems to floral meristems. Researchers have combined a genetic approach with tissue transplantation to study induction in the development of tomato flowers. As shown in<span class="SmallCap">FIGURE 21.19a</span>(p. 422), a floral meristem is a "bump" consisting of three layers of cells (L1-L3). All three layers participate in the formation of a flower, a reproductive structure with four types of organs: carpels (containing egg cells), petals, stamens (containing sperm-bearing pollen), and sepals (leaflike structures outside the petals).</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig21-19','../fig/big/fig21-19.jpg','fig21-19');"><img border="0" src="../fig/thm/fig21-19.jpg" alt="fig21-19" name="myfig21-19"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 21-19.</B>&nbsp;<span><span class="Emphasis2">Induction in flower development.</span></span></td></tr></table><a name="para05"></a><p class="BlockText">Researchers discovered that tomato plants homozygous for a mutant allele called<span class="Emphasis1">fasciated</span><span class="Emphasis1">(f)</span>produce flowers with an abnormally large number of organs. Taking advantage of the totipotency of plant cells, they performed an experiment in which they grafted stems from this mutant onto wild-type plants and then grew new plants from the shoots that emerged at the graft sites. The new plants were<span class="Emphasis2">chimeras,</span>organisms with a mixture of genetically different cells. Some of the chimeras produced floral meristems in which the three cell layers did not all come from the same "parent" (<span class="SmallCap">FIGURE 21.19b</span>). The researchers could identify the parental sources of the meristem layers by monitoring other genetic markers, such as an unrelated mutation that caused yellow leaves. They found that the number of organs per flower depends on the genes of the L3 (innermost) cell layer, which somehow induces the overlaying layers L2 and L1 to form that number of organs. The mechanism of cell-cell signaling leading to this induction is not yet known.</p><p class="Header3">Organ Identity Genes in Plants</p><a name="para06"></a><p class="BlockText">In contrast to genes controlling organ<span class="Emphasis1">number</span>in flowers are genes controlling organ<span class="Emphasis1">identity</span>. An<span class="Emphasis2">organ identity gene</span>determines the type of structure that will grow from a meristem--for instance, whether a particular outgrowth from a floral meristem becomes a petal or a stamen. Most of what we know about plant organ identity genes comes from research on flower development in<span class="Emphasis1">Arabidopsis</span>. Organ identity genes are analogous to homeotic genes in animals and are often referred to as plant homeotic genes. Just as a mutation in a fruit fly homeotic gene can cause legs to grow in place of antennae, a mutation in a plant organ identity gene can cause carpels to grow in place of sepals.</p><a name="para07"></a><p class="BlockText">By collecting and studying mutants with abnormal flowers, researchers have been able to identify and clone a number of floral organ identity genes, and they are beginning to determine how they act. To follow their work, we need a top view of the floral meristem (<span class="SmallCap">FIGURE 21.20a</span>). Viewed from above, the meristem can be divided into four concentric circles, or whorls, each of which develops into a circle of identical organs. The organ-identity genes first identified fell into three classes (<span class="Emphasis1">A</span>,<span class="Emphasis1">B</span>, and<span class="Emphasis1">C</span>), each of which affects two adjacent whorls. A simple model, diagrammed in<span class="SmallCap">FIGURE 21.20a</span>, explains how these three kinds of genes can direct the formation of the four types of organs.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig21-20a','../fig/big/fig21-20a.jpg','fig21-20a');"><img border="0" src="../fig/thm/fig21-20a.jpg" alt="fig21-20a" name="myfig21-20a"></a><br>a.</td></tr><tr><td valign="top" class="text"><a href="javascript:bigFig('fig21-20b','../fig/big/fig21-20b.jpg','fig21-20b');"><img border="0" src="../fig/thm/fig21-20b.jpg" alt="fig21-20b" name="myfig21-20b"></a><br>b.</td></tr><tr><td valign="top" class="text"><a href="javascript:bigFig('fig21-20c','../fig/big/fig21-20c.jpg','fig21-20c');"><img border="0" src="../fig/thm/fig21-20c.jpg" alt="fig21-20c" name="myfig21-20c"></a><br>c.</td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 21-20.</B>&nbsp;<span><span class="Emphasis2">Organ-identity genes and pattern formation in flower development.</span>This is a model for how genes specify the floral organs in<span class="Emphasis1">Arabidopsis</span>, with some supporting evidence.</span></td></tr></table><a name="para08"></a><p class="BlockText">This model (hypothesis) predicts that each organ identity gene will be transcribed in two specific whorls of the floral meristem, and once the genes were cloned, the predictions could be tested. Using nucleic acid from cloned genes as probes, researchers showed that the mRNA resulting from the transcription of each class of organ identity gene is present in the appropriate whorls of the developing floral meristem. For example, nucleic acid from a<span class="Emphasis1">C</span>gene hybridized appreciably only to cells in whorls 3 and 4 because only those cells were making RNA complementary to it (<span class="SmallCap">FIGURE 21.20b</span>).</p><a name="para09"></a><p class="BlockText">The model also can account for the phenotypes of mutants lacking<span class="Emphasis1">A</span>,<span class="Emphasis1">B</span>, or<span class="Emphasis1">C</span>gene activity with one addition: Where<span class="Emphasis1">A</span>gene activity is present, it inhibits<span class="Emphasis1">C</span>, and vice versa; if either<span class="Emphasis1">A</span>or<span class="Emphasis1">C</span>is missing, the other, not inhibited, takes its place.<span class="SmallCap">FIGURE 21.20c</span>shows the floral patterns of mutants lacking each of the three classes of organ identity genes and explains how the model accounts for the floral phenotypes.</p><a name="para10"></a><p class="BlockText">Further testing of this model and other hypotheses led in 2001 to the discovery of a fourth class of organ identity genes, termed class<span class="Emphasis1">E</span>. This discovery, combined with studies of the genes and gene products at the molecular level, is allowing researchers to extend and refine the older model.</p><a name="para11"></a><p class="BlockText">Presumably the organ identity genes are acting as master regulatory genes, each controlling the activity of a battery of other genes that more directly bring about an organ’s structure and function. Indeed, like the homeotic genes of animals, the organ identity genes of plants encode transcription factors that regulate other genes by binding to specific enhancers or promoters in the DNA. The plant genes, however, do not contain the homeobox DNA sequence that characterizes homeotic genes in animals. Instead, they have a sequence that encodes a different DNA-binding domain. Possibly as ancient as the homeobox, this sequence is also present in some transcription factor genes in yeast and animals.</p><a name="para12"></a><p class="BlockText">The more scientists learn about the molecules and mechanisms of development in different organisms, the more striking are the underlying similarities that emerge. The similarities reflect the unity--the common ancestry--of life on Earth. But the differences are also crucial, for they have created the huge diversity of organisms that have evolved.</p><a name="para13"></a><p class="BlockText">The remainder of the book goes beyond molecules, cells, and genes to explore life on the organismal level.</p></td></tr></table></body></html>
