<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;Genes can be cloned in recombinant DNA vectors: <span class="Emphasis1">a closer look</span></p><a name="para01"></a><p class="BlockText">With a restriction enzyme and DNA ligase in our toolkit, we can make the recombinant plasmid depicted in step 2 of <span class="SmallCap">FIGURE 20.1</span>. The original plasmid is called a <span class="Emphasis2">cloning vector,</span> defined as a DNA molecule that can carry foreign DNA into a cell and replicate there. Bacterial plasmids are widely used as cloning vectors. Recombinant plasmids produced by splicing restriction fragments from foreign DNA into plasmids isolated from bacteria can be returned relatively easily to bacteria. Then, as a bacterium carrying a recombinant plasmid reproduces, the plasmid replicates within it. The resulting cell clone, which appears as a colony on solid nutrient medium, contains multiple copies--a "clone"--of the foreign DNA.</p><a name="para02"></a><p class="BlockText">Bacteria are the most commonly used host cells for gene cloning, primarily because of the ease with which DNA can be isolated from and reintroduced into such cells. Bacterial cultures also grow quickly, rapidly replicating any foreign genes they carry.</p><p class="Header3">Procedure for Cloning a Eukaryotic Gene in a Bacterial Plasmid</p><a name="para03"></a><p class="BlockText">Suppose we want to clone a particular eukaryotic gene using the method sketched out in steps 1-5 of <span class="SmallCap">FIGURE 20.1</span>. The more detailed diagram in <span class="SmallCap">FIGURE 20.3</span> shows one way we might proceed. The step numbers in the text that follows correspond to the numbers in the figure.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-3','../fig/big/fig20-3.jpg','fig20-3');"><img border="0" src="../fig/thm/fig20-3.jpg" alt="fig20-3" name="myfig20-3"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-3.</B>&nbsp;<span><span class="Emphasis2">Cloning a human gene in a bacterial plasmid: a closer look.</span> In this figure, double-stranded DNA is represented as a single band. (Refer to <span class="SmallCap">FIGURE 20.2</span> to see steps 2a-2c in more detail.) The circled numbers correspond to numbered steps in the text. The diagram focuses on a fragment of human DNA carrying a gene of interest (black), but steps 2a-2c actually produce an enormous variety of recombinant plasmids, each of which shows up as a separate colony in step 4. The method used to identify the clones of interest (step 4b) relies on the <span class="Emphasis1">amp</span><SUP><span class="Emphasis1">R</span></SUP> and <span class="Emphasis1">lacZ</span> genes of the vector used here; other vectors and identification methods can also be used. Identifying the desired colony, step 5, is often the greatest challenge in this "shotgun" procedure.</span></td></tr></table><a name="para04"></a><p class="BlockText">
			&nbsp;<img align="middle" src="../img/sym/disk1.gif">&nbsp;
		<span class="Emphasis2">Isolation of vector and gene-source DNA.</span> We begin by preparing two kinds of DNA: the bacterial plasmid we’ll use as a vector and DNA containing the gene of interest. This latter DNA comes from human tissue cells that we have grown in laboratory culture. The plasmid comes from the bacterium <span class="Emphasis1">E. coli</span> and carries two genes that will later prove useful: <span class="Emphasis1">amp</span><SUP><span class="Emphasis1">R</span></SUP>, conferring resistance to the antibiotic ampicillin on its <span class="Emphasis1">E. coli</span> host cell, and <span class="Emphasis1">lacZ,</span> encoding the enzyme <span class="symbol">b</span>-galactosidase, which catalyzes the hydrolysis of the sugar lactose. The plasmid has a single recognition sequence for the restriction enzyme used, and the sequence lies within the <span class="Emphasis1">lacZ</span> gene.</p><a name="para05"></a><p class="BlockText">
			&nbsp;<img align="middle" src="../img/sym/disk2.gif">&nbsp;
		<span class="Emphasis2">Insertion of DNA into the vector.</span> In step 2a, we digest both the plasmid and the human DNA with the same restriction enzyme. The enzyme cuts the plasmid DNA at its single restriction site, disrupting the <span class="Emphasis1">lacZ</span> gene. It also cuts the human DNA, generating many thousands of fragments; one of these fragments carries the gene we want. In making the cuts, the restriction enzyme creates compatible sticky ends on both the human DNA fragments and the plasmid. For simplicity, <span class="SmallCap">FIGURE 20.3</span> shows the step-by-step processing of one human DNA fragment and one plasmid; but actually, millions of copies of the plasmid and a heterogeneous mixture of millions of human DNA fragments are treated simultaneously.</p><a name="para06"></a><p class="BlockText">In step 2b, we mix the fragments of human DNA with the clipped plasmids. The sticky ends of a plasmid base-pair with the complementary sticky ends of the human DNA fragment we are following. (Many other combinations form as well, such as two plasmids together or a plasmid with several DNA fragments; and re-formed, nonrecombinant plasmids are also common.) In step 2c, we use the enzyme DNA ligase to join the DNA molecules by covalent bonds. The result is a mixture of recombinant DNA molecules, some of which are like the one shown.</p><a name="para07"></a><p class="BlockText">
			&nbsp;<img align="middle" src="../img/sym/disk3.gif">&nbsp;
		<span class="Emphasis2">Introduction of the cloning vector into cells.</span> In this step, bacterial cells take up the recombinant plasmids by transformation (the uptake of naked DNA from the surrounding solution; see pp. 341-342). The bacteria are <span class="Emphasis1">lacZ</span><SUP>-</SUP> a mutation in gene <span class="Emphasis1">lacZ</span> makes them unable to hydrolyze lactose. Some of the bacteria acquire the desired recombinant plasmid DNA; many other cells take up other DNA, both recombinant and nonrecombinant.</p><a name="para08"></a><p class="BlockText">
			&nbsp;<img align="middle" src="../img/sym/disk4.gif">&nbsp;
		<span class="Emphasis2">Cloning of cells (and foreign genes).</span> Finally, we arrive at the actual cloning step. We plate out the transformed bacteria on solid nutrient medium containing ampicillin and a sugar called X-gal (step 4a). Each reproducing bacterium forms a cell clone that appears as a colony on the nutrient medium. In the process, any human genes carried by recombinant plasmids are also cloned. We take advantage of the plasmid’s own genes to select colonies of cells that carry recombinant plasmids. The ampicillin in the medium ensures that only plasmid-containing cells grow, for only they have the <span class="Emphasis1">amp</span><SUP><span class="Emphasis1">R</span></SUP> gene conferring ampicillin resistance. The X-gal in the medium makes it easy to identify colonies of bacteria whose plasmids carry foreign DNA. X-gal is hydrolyzed by <span class="symbol">b</span>-galactosidase to yield a blue product, so bacterial colonies containing plasmids with intact <span class="symbol">b</span>-galactosidase genes will be blue. But if a plasmid has foreign DNA inserted into its <span class="Emphasis1">lacZ</span> gene, then a colony of cells containing it will be white, because the cells cannot produce <span class="symbol">b</span>-galactosidase. In summary, with this method, bacteria with recombinant plasmids carrying foreign DNA will form white colonies on medium containing ampicillin and X-gal (step 4b).</p><a name="para09"></a><p class="BlockText">Our procedure to this point will have cloned many different human DNA fragments, not just the one that interests us. The next step, screening colonies for the desired human gene, is the most difficult.</p><a name="para10"></a><p class="BlockText">
			&nbsp;<img align="middle" src="../img/sym/disk5.gif">&nbsp;
		<span class="Emphasis2">Identification of cell clones carrying the gene of interest.</span> How can we distinguish the colony containing our gene of interest from the many thousands of colonies carrying other pieces of human DNA? We can look either for the gene itself or for its protein product. All the methods for detecting the DNA of a gene directly depend on base pairing between the gene and a complementary sequence on another nucleic acid molecule, a process called <span class="Emphasis2">nucleic acid hybridization.</span> The complementary molecule, a short, single-stranded nucleic acid that can be either RNA or DNA, is called a <span class="Emphasis2">nucleic acid probe.</span> If we know at least part of the nucleotide sequence of our gene (perhaps from knowledge of the protein it encodes), we can synthesize a probe complementary to it.</p><a name="para11"></a><p class="BlockText">We trace the probe, which will hydrogen-bond specifically to complementary single strands of the desired gene, by labeling it with a radioactive isotope or a fluorescent tag. <span class="SmallCap">FIGURE 20.4</span> (p. 380) shows how a number of bacterial clones, growing as colonies on a plate of solid medium (agar), can be simultaneously screened for the presence of DNA complementary to a DNA probe. An essential step is the <span class="Emphasis2">denaturation</span> of the cells’ DNA--that is, the separation of its two strands. As with protein denaturation, DNA denaturation is routinely accomplished with chemicals or heat. The labeled probe tags the colonies carrying the targeted gene.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-4','../fig/big/fig20-4.jpg','fig20-4');"><img border="0" src="../fig/thm/fig20-4.jpg" alt="fig20-4" name="myfig20-4"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-4.</B>&nbsp;<span><span class="Emphasis2">Using a nucleic acid probe to identify a cloned gene.</span> This technique depends on the fact that complementary nucleotide sequences will base-pair, stuck together by hydrogen bonds. Here the cloned gene of interest is carried on a bacterial plasmid, and the probe is a short length of radioactive single-stranded DNA complementary to part of the gene. 
			&nbsp;<img align="middle" src="../img/sym/disk1.gif">&nbsp;
		 Bacterial colonies on agar are pressed against special filter paper, transferring cells to the filter. 
			&nbsp;<img align="middle" src="../img/sym/disk2.gif">&nbsp;
		 The filter is treated to break open the cells and denature their DNA; the resulting single-stranded DNA molecules stick to the filter. 
			&nbsp;<img align="middle" src="../img/sym/disk3.gif">&nbsp;
		 A solution of probe molecules is incubated with the filter. The probe DNA hybridizes (base-pairs) with any complementary DNA on the filter; excess DNA is rinsed off. 
			&nbsp;<img align="middle" src="../img/sym/disk4.gif">&nbsp;
		 The filter is laid on photographic film, allowing any radioactive areas to expose the film (autoradiography). 
			&nbsp;<img align="middle" src="../img/sym/disk5.gif">&nbsp;
		 The developed film, an autoradiograph, is compared with the master culture plate to determine which colonies carry the desired gene.</span></td></tr></table><a name="para12"></a><p class="BlockText">Once we’ve identified a cell clone carrying the desired gene, we can grow the cells in liquid culture in a large tank and then easily isolate large amounts of the gene. Also, we can use the cloned gene itself as a probe to identify similar or identical genes in DNA from other sources.</p><a name="para13"></a><p class="BlockText">If the cells containing a desired gene translate the gene into protein, then it is sometimes possible to identify them by screening all clones for the protein. Detection of the protein can be based on either its activity (as with an enzyme) or its structure, using antibodies that bind specifically to it. However, for bacterial cells to make a eukaryotic protein, we need to take special measures, as discussed next.</p><p class="Header3">Cloning and Expressing Eukaryotic Genes: Problems and Solutions</p><a name="para14"></a><p class="BlockText">Getting a cloned eukaryotic gene to function in a prokaryotic setting can be difficult because certain details of gene expression are different in the two kinds of cells. To overcome differences in promoters and other DNA control sequences, scientists usually employ an <span class="Emphasis2">expression vector,</span> a cloning vector that contains the requisite prokaryotic promoter just upstream of a restriction site where the eukaryotic gene can be inserted. The bacterial host cell will then recognize the promoter and proceed to express the foreign gene that has been linked to it. Expression vectors allow the synthesis of many eukaryotic proteins in bacterial cells.</p><a name="para15"></a><p class="BlockText">Another problem with cloning and expressing eukaryotic DNA in bacteria is the presence of long noncoding regions (introns) in most eukaryotic genes. Introns can make a eukaryotic gene very long and unwieldy, and they prevent correct expression of the gene by bacterial cells, which do not have RNA-splicing machinery. Fortunately, we can make artificial eukaryotic genes that lack introns (<span class="SmallCap">FIGURE 20.5</span>). The starting material for this procedure is fully processed mRNA, which has been stripped of introns in the eukaryotic cell nucleus. We can extract this mRNA from cells and then use the enzyme reverse transcriptase (obtained from retroviruses) to make DNA transcripts of this RNA. Each DNA molecule produced carries the complete coding sequence for a gene but no introns. This DNA is called <span class="Emphasis2">complementary DNA,</span> or <span class="Emphasis2">cDNA,</span> and can be attached to vector DNA for replication inside a cell. Bacteria can express a eukaryotic cDNA gene if the vector provides a bacterial promoter and any other control elements necessary for the gene’s transcription and translation.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig20-5','../fig/big/fig20-5.jpg','fig20-5');"><img border="0" src="../fig/thm/fig20-5.jpg" alt="fig20-5" name="myfig20-5"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 20-5.</B>&nbsp;<span><span class="Emphasis2">Making complementary DNA (cDNA) for a eukaryotic gene.</span> Complementary DNA is DNA made <span class="Emphasis1">in vitro</span> using mRNA as a template and the enzyme reverse transcriptase. The synthesis and splicing of an mRNA molecule in a cell nucleus is shown in steps 
			&nbsp;<img align="middle" src="../img/sym/disk1.gif">&nbsp;
		 and 
			&nbsp;<img align="middle" src="../img/sym/disk2.gif">&nbsp;
		. In step 
			&nbsp;<img align="middle" src="../img/sym/disk3.gif">&nbsp;
		, we isolate the mRNA molecules from the cell and add reverse transcriptase, which makes a DNA strand using the RNA as a template. We provide 
			&nbsp;<img align="middle" src="../img/sym/disk4.gif">&nbsp;
		 another enzyme to degrade the RNA and 
			&nbsp;<img align="middle" src="../img/sym/disk5.gif">&nbsp;
		 DNA polymerase to synthesize a second DNA strand. The result is cDNA, which carries the complete coding sequence of the gene but no introns.</span></td></tr></table><a name="para16"></a><p class="BlockText">Molecular biologists can avoid eukaryotic-prokaryotic incompatibility by using eukaryotic cells rather than bacteria as hosts for cloning and/or expressing eukaryotic genes of interest. Yeast cells, single-celled fungi, offer two advantages: They are as easy to grow as bacteria, and they have plasmids, a rarity among eukaryotes. Scientists have even constructed recombinant plasmids that combine yeast and bacterial DNA and can replicate in either type of cell. Scientists have also constructed vectors called <span class="Emphasis2">yeast artificial chromosomes (YACs)</span> that combine the essentials of a eukaryotic chromosome--an origin for DNA replication, a centromere, and two telomeres--with foreign DNA. These chromosomes behave normally in mitosis, cloning the foreign DNA as the yeast cell divides. A YAC can carry much more DNA than can a plasmid vector, enabling very long pieces of DNA to be cloned.</p><a name="para17"></a><p class="BlockText">Another reason to use eukaryotic host cells for expressing a cloned eukaryotic gene is that many eukaryotic proteins will not function unless they are modified after translation, for example by the addition of carbohydrate or lipid groups. Bacterial cells cannot carry out these modifications, and if the gene product requiring such processing is from a mammal, even yeast cells will not be able to modify the protein correctly. The use of host cells from an animal or plant cell culture may therefore be necessary.</p><a name="para18"></a><p class="BlockText">Like bacteria, many kinds of eukaryotic cells can take up DNA from their surroundings but sometimes not very efficiently. To deal with this problem, scientists have developed a variety of more aggressive methods for introducing recombinant DNA into cells. In <span class="Emphasis2">electroporation,</span> they apply a brief electrical pulse to a solution containing cells. The electricity creates a temporary hole in the cell’s plasma membrane, through which DNA can enter. (This technique is now commonly used for bacteria as well.) Alternatively, scientists can inject DNA directly into single eukaryotic cells using microscopically thin needles. And in a technique used primarily for plant cells, they can attach DNA to microscopic particles of metal and fire the particles into cells with a gun (see <span class="SmallCap">FIGURE 38.17</span>). Once inside the cell, the DNA has a chance to be incorporated into the cell’s DNA by natural genetic recombination.</p></td></tr></table></body></html>
