<html><head>
<META http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"><title>Campbell Biology - Sixth Edition</title><script src="../js/scripts.js" language="JavaScript"></script></head><body bgcolor="#ffffff"><table width="500" border="0" cellspacing="0" cellpadding="0"><tr><td><p class="Header2"><img src="../img/bullet.gif" align="middle" border="0">&nbsp;The control of gene expression enables individual bacteria to adjust their metabolism to environmental change</p><a name="para01"></a><p class="BlockText">Mutations and the various types of genetic transfer we have been studying generate the genetic variation that makes natural selection possible. And natural selection, acting over many generations, can increase the proportion of individuals in a bacterial population that are adapted to some new environmental condition. But how can an individual bacterium, locked into the genome it has inherited, cope with environmental fluctuation?</p><a name="para02"></a><p class="BlockText">Think, for instance, of an <span class="Emphasis1">E. coli</span> cell living in the erratic environment of a human colon, dependent for its nutrients on the whimsical eating habits of its host. If the bacterium is deprived of the amino acid tryptophan, which it needs to survive, it responds by activating a metabolic pathway to make its own tryptophan from another compound. Later, if the human host eats a tryptophan-rich meal, the bacterial cell stops producing tryptophan for itself, thus saving the cell from squandering its resources to produce a substance that is available from the surrounding solution in prefabricated form. This is just one example of how bacteria tune their metabolism to changing environments.</p><a name="para03"></a><p class="BlockText">Metabolic control occurs on two levels (<span class="SmallCap">FIGURE 18.19</span>). First, cells can vary the numbers of specific enzyme molecules made; that is, they can regulate the expression of genes. Second, cells can adjust the activity of enzymes already present. The latter mode of control, which is more immediate, depends on the sensitivity of many enzymes to chemical cues that increase or decrease their catalytic activity (see Chapter 6). For example, activity of the first enzyme of the tryptophan synthesis pathway is inhibited by the pathway’s end product. Thus, if tryptophan accumulates in a cell, it shuts down its own synthesis. Such feedback inhibition, typical of anabolic (biosynthetic) pathways, allows a cell to adapt to short-term fluctuations in levels of a substance it needs.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-19','../fig/big/fig18-19.jpg','fig18-19');"><img border="0" src="../fig/thm/fig18-19.jpg" alt="fig18-19" name="myfig18-19"></a><br></td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-19.</B>&nbsp;<span><span class="Emphasis2">Regulation of a metabolic pathway.</span> In the pathway for tryptophan synthesis, an abundance of tryptophan can both <span class="Emphasis2">(a)</span> repress expression of the genes for all the enzymes needed for the pathway, and <span class="Emphasis2">(b)</span> inhibit the activity of the first enzyme in the pathway (feedback inhibition). The symbol  stands for "inhibition."</span></td></tr></table><a name="para04"></a><p class="BlockText">If, in our example, the environment continues to provide all the tryptophan the cell needs, the regulation of gene expression also comes into play: The cell stops making enzymes of the tryptophan pathway. This control of enzyme production occurs at the level of transcription, the synthesis of messenger RNA coding for these enzymes. More generally, many genes of the bacterial genome are switched on or off by changes in the metabolic status of the cell. The basic mechanism for this control of gene expression in bacteria, described as the operon model, was discovered in 1961 by Franois Jacob and Jacques Monod at the Pasteur Institute in Paris. Let’s see what an operon is and how it works, using the control of tryptophan synthesis as our first example.</p><p class="Header3">Operons: The Basic Concept</p><a name="para05"></a><p class="BlockText"><span class="Emphasis1">E. coli</span> synthesizes tryptophan from a precursor molecule in a series of steps, each reaction catalyzed by a specific enzyme (see <span class="SmallCap">FIGURE 18.19</span>). The five genes coding for the polypeptide chains that make up these enzymes are clustered together on the bacterial chromosome. A single promoter serves all five genes, which constitute a transcription unit. (Recall from Chapter 17 that a promoter is a site where RNA polymerase can bind to DNA and begin transcribing genes.) Thus, transcription gives rise to one long mRNA molecule representing all five genes for the tryptophan pathway. The cell can translate this transcript into separate polypeptides because the mRNA is punctuated with start and stop codons signaling where the coding sequence for each polypeptide begins and ends.</p><a name="para06"></a><p class="BlockText">A key advantage of grouping genes of related function into one transcription unit is that a single "on-off switch" can control the whole cluster of functionally related genes. When an <span class="Emphasis1">E. coli</span> cell must make tryptophan for itself because the nutrient medium lacks this amino acid, all the enzymes for the metabolic pathway are synthesized at one time. The switch is a segment of DNA called an <span class="Emphasis2">operator.</span> Both its location and name suit its function: Positioned within the promoter or between the promoter and the enzyme-coding genes, the operator controls the access of RNA polymerase to the genes. All together, the operator, the promoter, and the genes they control--the entire stretch of DNA required for enzyme production for the tryptophan pathway--is called an <span class="Emphasis2">operon</span> (<span class="SmallCap">FIGURE 18.20</span>). Here we are dissecting one of many operons that have been discovered in <span class="Emphasis1">E. coli</span>: the <span class="Emphasis1">trp</span> operon (<span class="Emphasis1">trp</span> for tryptophan).</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-20a','../fig/big/fig18-20a.jpg','fig18-20a');"><img border="0" src="../fig/thm/fig18-20a.jpg" alt="fig18-20a" name="myfig18-20a"></a><br>a.</td></tr><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-20b','../fig/big/fig18-20b.jpg','fig18-20b');"><img border="0" src="../fig/thm/fig18-20b.jpg" alt="fig18-20b" name="myfig18-20b"></a><br>b.</td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-20.</B>&nbsp;<span><span class="Emphasis2">The <span class="Emphasis1">trp</span> operon: regulated synthesis of repressible enzymes.</span> Tryptophan is an amino acid produced by an anabolic path-way catalyzed by repressible enzymes. Five genes encoding the polypeptides that make up the enzymes of this pathway are grouped into an operon, along with a promoter and an operator. (The <span class="Emphasis1">trp</span> operator is located within the <span class="Emphasis1">trp</span> promoter.) Accumulation of tryptophan, the end product of the pathway, represses synthesis of the enzymes. The mechanism in <span class="Emphasis1">E. coli</span> is shown here.</span></td></tr></table><a name="para07"></a><p class="BlockText">If the operator is the control point for transcription, what determines whether the operator is in the on or off mode? By itself, the operator is on; RNA polymerase can bind to the promoter and transcribe the genes of the operon. The operon can be switched off by a protein called the <span class="Emphasis2">repressor.</span> The repressor binds to the operator and blocks attachment of RNA polymerase to the promoter, preventing transcription of the genes. Repressor proteins are specific; that is, they recognize and bind only to the operator of a certain operon. The repressor that switches off the <span class="Emphasis1">trp</span> operon has no effect on other operons in the <span class="Emphasis1">E. coli</span> genome.</p><a name="para08"></a><p class="BlockText">The repressor is the product of a gene called a <span class="Emphasis2">regulatory gene.</span> The regulatory gene encoding the <span class="Emphasis1">trp</span> repressor, <span class="Emphasis1">trpR,</span> is located some distance away from the operon it controls and has its own promoter. Transcription of <span class="Emphasis1">trpR</span> produces an mRNA molecule that is translated into the repressor protein, which can then reach the operator of the <span class="Emphasis1">trp</span> operon by diffusion. Regulatory genes are transcribed continuously, although at a low rate, and a few <span class="Emphasis1">trp</span> repressor molecules are always present in the cell. Why, then, is the <span class="Emphasis1">trp</span> operon not switched off permanently? First, the binding of repressors to operators is reversible. An operator vacillates between the on and off modes, with the relative duration of each state depending on the number of active repressor molecules around. Secondly, the <span class="Emphasis1">trp</span> repressor, like most regulatory proteins, is an allosteric protein, with two alternative shapes, active and inactive (see <span class="SmallCap">FIGURE 6.18</span>). The <span class="Emphasis1">trp</span> repressor is synthesized in an inactive form with little affinity for the <span class="Emphasis1">trp</span> operator. Only if tryptophan binds to the repressor at an allosteric site does the repressor protein change to the active form that can attach to the operator, turning the operon off.</p><a name="para09"></a><p class="BlockText">Tryptophan functions in this system as a <span class="Emphasis2">corepressor,</span> a small molecule that cooperates with a repressor protein to switch an operon off. As tryptophan accumulates, more tryptophan molecules associate with <span class="Emphasis1">trp</span> repressor molecules, which can then bind to the <span class="Emphasis1">trp</span> operator and shut down tryptophan production. If the cell’s tryptophan level drops, transcription of the operon’s genes resumes. This is one example of how gene expression responds rapidly to changes in the cell’s internal and external environment.</p><p class="Header3">Repressible Versus Inducible Operons: Two Types of Negative Gene Regulation</p><a name="para10"></a><p class="BlockText">The <span class="Emphasis1">trp</span> operon is said to be a <span class="Emphasis1">repressible operon</span> because its transcription is <span class="Emphasis1">inhibited</span> when a specific small molecule (tryptophan) binds allosterically to a regulatory protein. In contrast, an <span class="Emphasis1">inducible operon</span> is <span class="Emphasis1">stimulated</span> (that is, induced) when a specific small molecule interacts with a regulatory protein. Let’s investigate an example, which was actually the operon first worked out by Jacob and Monod (<span class="SmallCap">FIGURE 18.21</span>).</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-21a','../fig/big/fig18-21a.jpg','fig18-21a');"><img border="0" src="../fig/thm/fig18-21a.jpg" alt="fig18-21a" name="myfig18-21a"></a><br>a.</td></tr><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-21b','../fig/big/fig18-21b.jpg','fig18-21b');"><img border="0" src="../fig/thm/fig18-21b.jpg" alt="fig18-21b" name="myfig18-21b"></a><br>b.</td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-21.</B>&nbsp;<span><span class="Emphasis2">The <span class="Emphasis1">lac</span> operon: regulated synthesis of inducible enzymes.</span>&nbsp;<span class="Emphasis1">E. coli</span> uses three enzymes to take up and metabolize lactose. The genes for these three enzymes are clustered in the <span class="Emphasis1">lac</span> operon. One gene, <span class="Emphasis1">lacZ,</span> codes for <span class="symbol">b</span>-galactosidase, which hydrolyzes lactose to glucose and galactose. The second gene, <span class="Emphasis1">lacY,</span> codes for a permease, the membrane protein that transports lactose into the cell. The third gene, <span class="Emphasis1">lacA,</span> codes for an enzyme called transacetylase, whose function in lactose metabolism is still unclear. The gene for the <span class="Emphasis1">lac</span> repressor, <span class="Emphasis1">lacI</span>, happens to be adjacent to the <span class="Emphasis1">lac</span> operon, an unusual situation. (The function of the upstream (left) end of the promoter is revealed in <span class="SmallCap">FIGURE 18.22</span>.)</span></td></tr></table><a name="para11"></a><p class="BlockText">The disaccharide lactose (milk sugar) is available to <span class="Emphasis1">E. coli</span> if the human host drinks milk. The bacteria can absorb the lactose and break it down for energy or use it as a source of organic carbon for synthesizing other compounds. Lactose metabolism begins with hydrolysis of the disaccharide into its two component monosaccharides, glucose and galactose. The enzyme that catalyzes this reaction is called <span class="symbol">b</span>-galactosidase. Only a few molecules of this enzyme are present in an <span class="Emphasis1">E. coli</span> cell that has been growing in the absence of lactose--in the intestines of a person who does not drink milk, for example. But if lactose is added to the bacterium’s nutrient medium, it takes only about 15 minutes for the number of <span class="symbol">b</span>-galactosidase molecules in the cell to increase a thousandfold.</p><a name="para12"></a><p class="BlockText">The gene for <span class="symbol">b</span>-galactosidase is part of an operon, the <span class="Emphasis1">lac</span> operon (<span class="Emphasis1">lac</span> for lactose metabolism), that includes two other genes coding for proteins that function in lactose metabolism (see <span class="SmallCap">FIGURE 18.21</span>). This entire transcription unit is under the command of a single operator and promoter. The regulatory gene, <span class="Emphasis1">lacI,</span> located outside the operon, codes for an allosteric repressor protein that can switch off the <span class="Emphasis1">lac</span> operon by binding to the operator. So far, this sounds just like regulation of the <span class="Emphasis1">trp</span> operon, but there is one important difference. Recall that the <span class="Emphasis1">trp</span> repressor was innately inactive and required tryptophan as a corepressor in order to bind to the operator. The <span class="Emphasis1">lac</span> repressor, in contrast, is active all by itself, binding to the operator and switching the <span class="Emphasis1">lac</span> operon off. In this case, a specific small molecule, called an <span class="Emphasis2">inducer,</span>&nbsp;<span class="Emphasis1">inactivates</span> the repressor. For the <span class="Emphasis1">lac</span> operon, the inducer is allolactose, an isomer of lactose formed in small amounts from lactose that enters the cell. In the absence of lactose (and hence allolactose), the <span class="Emphasis1">lac</span> repressor is in its active configuration, and the genes of the <span class="Emphasis1">lac</span> operon are silenced. If lactose is added to the cell’s nutrient medium, allolactose binds to the <span class="Emphasis1">lac</span> repressor and alters its conformation, nullifying the repressor’s ability to attach to the operator. Now, on demand, the <span class="Emphasis1">lac</span> operon produces mRNA for the enzymes of the lactose pathway. In the context of gene regulation, these enzymes are referred to as inducible enzymes, because their synthesis is induced by a chemical signal (allolactose, in this case). Analogously, the enzymes for tryptophan synthesis are said to be repressible.</p><a name="para13"></a><p class="BlockText">Let’s compare repressible enzymes and inducible enzymes in terms of the metabolic economy of the <span class="Emphasis1">E. coli</span> cell. Repressible enzymes generally function in anabolic pathways, which synthesize essential end products from raw materials (precursors). By suspending production of an end product when it is already present in sufficient quantity, the cell can allocate its organic precursors and energy for other uses. In contrast, inducible enzymes usually function in catabolic pathways, which break a nutrient down to simpler molecules. By producing the appropriate enzymes only when the nutrient is available, the cell avoids making proteins that have nothing to do. Why bother, for example, to make the enzymes that break down milk sugar when no milk is present?</p><a name="para14"></a><p class="BlockText">In comparing repressible and inducible enzymes, there is one more important point: Both systems are examples of the <span class="Emphasis1">negative</span> control of genes, because the operons are switched <span class="Emphasis1">off</span> by the active form of the repressor protein. It may be easier to see this in the case of the <span class="Emphasis1">trp</span> operon, but it is true for the <span class="Emphasis1">lac</span> operon as well. Allolactose induces enzyme synthesis not by acting directly on the genome, but by freeing the <span class="Emphasis1">lac</span> operon from the negative effect of the repressor. Technically, allolactose is more of a <span class="Emphasis1">derepressor</span> than an inducer of genes. Gene regulation is said to be positive only when an activator molecule interacts directly with the genome to switch transcription on. Let’s look at an example, again involving the <span class="Emphasis1">lac</span> operon.</p><p class="Header3">An Example of Positive Gene Regulation</p><a name="para15"></a><p class="BlockText">For the enzymes that break down lactose to be synthesized in appreciable quantity, it is not enough that lactose be present in the bacterial cell. The other requirement is that the simple sugar glucose be in short supply. Given a choice of substrates for glycolysis and other catabolic pathways, <span class="Emphasis1">E. coli</span> preferentially uses glucose, the sugar most reliably present in its environment. The enzymes for glucose breakdown (glycolysis; see <span class="SmallCap">FIGURE 9.9</span>) are continually present.</p><a name="para16"></a><p class="BlockText">How does the <span class="Emphasis1">E. coli</span> cell sense the glucose concentration, and how is this information relayed to the genome? Again, the mechanism depends on the interaction of an allosteric regulatory protein with a small organic molecule. The small molecule is <span class="Emphasis2">cyclic AMP (cAMP),</span> which accumulates when glucose is scarce (see <span class="SmallCap">FIGURE 11.12</span> for the structure of cAMP). The regulatory protein is <span class="Emphasis2">cAMP receptor protein (CRP),</span> and it is an <span class="Emphasis1">activator</span> of transcription. When cAMP binds to the allosteric site on CRP, the protein assumes its active shape and can bind to a specific site at the upstream end of the <span class="Emphasis1">lac</span> promoter (<span class="SmallCap">FIGURE 18.22</span>). The attachment of CRP actually bends the DNA, which somehow makes it easier for RNA polymerase to bind to the promoter and start transcription of the operon. Because CRP is a regulatory protein that directly stimulates gene expression, this mechanism qualifies as positive regulation.</p><table cellpadding="5" cellspacing="0" border="0"><tr><td><table boder="0" cellpadding="2" cellspacing="0"><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-22a','../fig/big/fig18-22a.jpg','fig18-22a');"><img border="0" src="../fig/thm/fig18-22a.jpg" alt="fig18-22a" name="myfig18-22a"></a><br>a.</td></tr><tr><td valign="top" class="text"><a href="javascript:bigFig('fig18-22b','../fig/big/fig18-22b.jpg','fig18-22b');"><img border="0" src="../fig/thm/fig18-22b.jpg" alt="fig18-22b" name="myfig18-22b"></a><br>b.</td></tr><tr><td align="center" style="font-size:9pt;"><i>Click image to enlarge</i></td></tr></table></td><td class="figure_caption" bgcolor="#f1f1f1" valign="top"><B>Fig 18-22.</B>&nbsp;<span><span class="Emphasis2">Positive control: cAMP receptor protein.</span> RNA polymerase has a low affinity for the promoter of the <span class="Emphasis1">lac</span> operon unless helped by a regulatory protein called the cAMP receptor protein (CRP), which binds to a DNA site at the upstream end of the promoter. The CRP molecule can attach to the DNA only when associated with cyclic AMP (cAMP), whose concentration in the cell rises when the glucose concentration falls. Thus, even if lactose is available, the cell preferentially catabolizes glucose, using enzymes that are always present. This regulatory system ensures that <span class="Emphasis1">E. coli</span> will gear up for consumption of lactose and other secondary catabolites only when glucose is scarce.</span></td></tr></table><a name="para17"></a><p class="BlockText">If the amount of glucose in the cell increases, the cAMP concentration falls, and without it, CRP disengages from the operon. Because CRP is inactive, transcription of the <span class="Emphasis1">lac</span> operon proceeds at only a low level, even in the presence of lactose. Thus, the <span class="Emphasis1">lac</span> operon is under dual control: negative control by the <span class="Emphasis1">lac</span> repressor and positive control by CRP. The state of the <span class="Emphasis1">lac</span> repressor (with or without allolactose) determines whether or not transcription of the <span class="Emphasis1">lac</span> operon’s genes can occur; the state of CRP (with or without cAMP) controls the rate of transcription if the operon is repressor-free. It is as though the operon has both an on-off switch and a volume control.</p><a name="para18"></a><p class="BlockText">Although we have used the <span class="Emphasis1">lac</span> operon as an example, CRP, unlike repressor proteins, works on several different operons that encode enzymes used in catabolic pathways. When glucose is present and CRP is inactive, there is a general slowdown in the synthesis of enzymes required for the catabolism of compounds other than glucose. The cell’s ability to catabolize other compounds, such as lactose, provides backup systems that enable a cell deprived of glucose to survive. The specific compounds present at the moment determine which operons are switched on. These elaborate contingency mechanisms suit an organism that cannot control what its host eats. Bacteria are remarkable in their ability to adapt--over the long term by evolutionary changes in their genetic makeup and over the short term by the control of gene expression in individual cells. Of course, the various control mechanisms are also evolutionary products that exist because they have been favored by natural selection.</p><a name="para19"></a><p class="BlockText">Molecular genetics was founded on the study of viruses and bacteria, the microbial models that have been the subjects of this chapter. Eukaryotic organisms and their genomes are much more complex, and only in the past decade or so have researchers begun to learn how the control of gene expression helps bring about this complexity. In the next chapter, we begin to explore this topic.</p></td></tr></table></body></html>
